<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>4 Analyzing data from an experiment | Portfolio</title>
  <meta name="description" content="4 Analyzing data from an experiment | Portfolio" />
  <meta name="generator" content="bookdown 0.30 and GitBook 2.6.7" />

  <meta property="og:title" content="4 Analyzing data from an experiment | Portfolio" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="4 Analyzing data from an experiment | Portfolio" />
  
  
  

<meta name="author" content="Nine Luijendijk" />


<meta name="date" content="2022-12-22" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="guerillaframework.html"/>
<link rel="next" href="primates.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Prerequisites</a></li>
<li class="chapter" data-level="2" data-path="resume.html"><a href="resume.html"><i class="fa fa-check"></i><b>2</b> Curriculum Vitae Nine Luijendijk</a>
<ul>
<li class="chapter" data-level="2.1" data-path="resume.html"><a href="resume.html#contact"><i class="fa fa-check"></i><b>2.1</b> Contact</a></li>
<li class="chapter" data-level="2.2" data-path="resume.html"><a href="resume.html#education"><i class="fa fa-check"></i><b>2.2</b> Education</a></li>
<li class="chapter" data-level="2.3" data-path="resume.html"><a href="resume.html#experience"><i class="fa fa-check"></i><b>2.3</b> Experience</a></li>
<li class="chapter" data-level="2.4" data-path="resume.html"><a href="resume.html#skills"><i class="fa fa-check"></i><b>2.4</b> Skills</a></li>
<li class="chapter" data-level="2.5" data-path="resume.html"><a href="resume.html#languages"><i class="fa fa-check"></i><b>2.5</b> Languages</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="guerillaframework.html"><a href="guerillaframework.html"><i class="fa fa-check"></i><b>3</b> Guerilla Analytics framework</a></li>
<li class="chapter" data-level="4" data-path="datanalysis.html"><a href="datanalysis.html"><i class="fa fa-check"></i><b>4</b> Analyzing data from an experiment</a>
<ul>
<li class="chapter" data-level="4.1" data-path="datanalysis.html"><a href="datanalysis.html#open-the-file-in-r-and-inspect-the-data."><i class="fa fa-check"></i><b>4.1</b> Open the file in R and inspect the data.</a></li>
<li class="chapter" data-level="4.2" data-path="datanalysis.html"><a href="datanalysis.html#fixing-the-characterdouble-issue"><i class="fa fa-check"></i><b>4.2</b> Fixing the character/double issue</a></li>
<li class="chapter" data-level="4.3" data-path="datanalysis.html"><a href="datanalysis.html#think-about-how-you-would-analyze-this-experiment-to-learn-whether-there-is-indeed-an-effect-of-different-concentrations-on-offspring-count-and-whether-the-different-compounds-have-a-different-curve-ic50.-write-down-you-analysis-as-a-step-wise-plan."><i class="fa fa-check"></i><b>4.3</b> Think about how you would analyze this experiment to learn whether there is indeed an effect of different concentrations on offspring count and whether the different compounds have a different curve (IC50). Write down you analysis as a step-wise plan.</a></li>
<li class="chapter" data-level="4.4" data-path="datanalysis.html"><a href="datanalysis.html#normalize-the-data-for-the-controlnegative-in-such-a-way-that-the-mean-value-for-controlnegative-is-exactly-equal-to-1-and-that-all-other-values-are-expressed-as-a-fraction-thereof.-rerun-your-graphs-with-the-normalized-data."><i class="fa fa-check"></i><b>4.4</b> Normalize the data for the controlNegative in such a way that the mean value for controlNegative is exactly equal to 1 and that all other values are expressed as a fraction thereof. Rerun your graphs with the normalized data.</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="primates.html"><a href="primates.html"><i class="fa fa-check"></i><b>5</b> Generating a phylogenetic tree in R</a>
<ul>
<li class="chapter" data-level="5.1" data-path="primates.html"><a href="primates.html#the-plan"><i class="fa fa-check"></i><b>5.1</b> The plan</a></li>
<li class="chapter" data-level="5.2" data-path="primates.html"><a href="primates.html#preparatory-research"><i class="fa fa-check"></i><b>5.2</b> Preparatory research</a></li>
<li class="chapter" data-level="5.3" data-path="primates.html"><a href="primates.html#getting-started"><i class="fa fa-check"></i><b>5.3</b> Getting started</a></li>
<li class="chapter" data-level="5.4" data-path="primates.html"><a href="primates.html#generating-the-first-tree"><i class="fa fa-check"></i><b>5.4</b> Generating the first tree</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="primates.html"><a href="primates.html#obtaining-fasta-sequences"><i class="fa fa-check"></i><b>5.4.1</b> Obtaining FASTA sequences</a></li>
<li class="chapter" data-level="5.4.2" data-path="primates.html"><a href="primates.html#performing-multiple-sequence-alignment"><i class="fa fa-check"></i><b>5.4.2</b> Performing multiple sequence alignment</a></li>
<li class="chapter" data-level="5.4.3" data-path="primates.html"><a href="primates.html#importing-the-aligned-dna"><i class="fa fa-check"></i><b>5.4.3</b> Importing the aligned dna</a></li>
<li class="chapter" data-level="5.4.4" data-path="primates.html"><a href="primates.html#calculating-the-genetic-distances-between-the-sequences-and-drawing-a-tree"><i class="fa fa-check"></i><b>5.4.4</b> Calculating the genetic distances between the sequences and drawing a tree</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="primates.html"><a href="primates.html#different-types-of-phylogenetic-models"><i class="fa fa-check"></i><b>5.5</b> Different types of phylogenetic models</a>
<ul>
<li class="chapter" data-level="5.5.1" data-path="primates.html"><a href="primates.html#finding-the-best-model"><i class="fa fa-check"></i><b>5.5.1</b> Finding the best model</a></li>
<li class="chapter" data-level="5.5.2" data-path="primates.html"><a href="primates.html#generating-a-maximum-likelihood-tree"><i class="fa fa-check"></i><b>5.5.2</b> Generating a maximum likelihood tree</a></li>
<li class="chapter" data-level="5.5.3" data-path="primates.html"><a href="primates.html#visualizing-the-maximum-likelihood-cladogram"><i class="fa fa-check"></i><b>5.5.3</b> Visualizing the maximum likelihood cladogram</a></li>
<li class="chapter" data-level="5.5.4" data-path="primates.html"><a href="primates.html#rooting-the-tree"><i class="fa fa-check"></i><b>5.5.4</b> Rooting the tree</a></li>
</ul></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Portfolio</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="datanalysis" class="section level1 hasAnchor" number="4">
<h1><span class="header-section-number">4</span> Analyzing data from an experiment<a href="datanalysis.html#datanalysis" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>In this experiment, the number of offspring of adult C. elegans was counted after exposing the nematodes to different substances in multiple concentrations. The data was supplied by J. Louter of the INT/ILC.</p>
<div id="open-the-file-in-r-and-inspect-the-data." class="section level2 hasAnchor" number="4.1">
<h2><span class="header-section-number">4.1</span> Open the file in R and inspect the data.<a href="datanalysis.html#open-the-file-in-r-and-inspect-the-data." class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="datanalysis.html#cb3-1" aria-hidden="true" tabindex="-1"></a>celegans_data <span class="ot">&lt;-</span> <span class="fu">read_excel</span>(<span class="st">&quot;/Users/nineluijendijk/Downloads/CE.LIQ.FLOW.062_Tidydata.xlsx&quot;</span>)</span></code></pre></div>
<p>Expected data types would be double/integer for RawData (number of offspring), character for compName (name of the compound) and double for compConcentration (concentration of the compound). The actual data types of the columns are double for RawData, character for compName and character for compConcentration. The data type for compConcentration has not been correctly assigned which will cause the following issue if it’s not changed:</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="datanalysis.html#cb4-1" aria-hidden="true" tabindex="-1"></a><span class="co">#Create a scatter plot</span></span>
<span id="cb4-2"><a href="datanalysis.html#cb4-2" aria-hidden="true" tabindex="-1"></a>plot_chr <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="at">data =</span> celegans_data, <span class="fu">aes</span>(<span class="at">x =</span> compConcentration, <span class="at">y =</span> RawData))<span class="sc">+</span></span>
<span id="cb4-3"><a href="datanalysis.html#cb4-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="fu">aes</span>(<span class="at">color =</span> compName,</span>
<span id="cb4-4"><a href="datanalysis.html#cb4-4" aria-hidden="true" tabindex="-1"></a>                 <span class="at">shape =</span> expType),</span>
<span id="cb4-5"><a href="datanalysis.html#cb4-5" aria-hidden="true" tabindex="-1"></a>             <span class="at">size =</span> <span class="dv">1</span>)<span class="sc">+</span></span>
<span id="cb4-6"><a href="datanalysis.html#cb4-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rotate_axis_labels</span>(<span class="st">&quot;x&quot;</span>, <span class="dv">90</span>)<span class="sc">+</span></span>
<span id="cb4-7"><a href="datanalysis.html#cb4-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">&quot;Concentration&quot;</span>,</span>
<span id="cb4-8"><a href="datanalysis.html#cb4-8" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">&quot;Number of offspring&quot;</span>,</span>
<span id="cb4-9"><a href="datanalysis.html#cb4-9" aria-hidden="true" tabindex="-1"></a>       <span class="at">title =</span> <span class="st">&quot;Number of C. elegans offspring under</span><span class="sc">\n</span><span class="st">a number of circumstances, alphabetical&quot;</span>,</span>
<span id="cb4-10"><a href="datanalysis.html#cb4-10" aria-hidden="true" tabindex="-1"></a>       <span class="at">shape =</span> <span class="st">&quot;Type&quot;</span>,</span>
<span id="cb4-11"><a href="datanalysis.html#cb4-11" aria-hidden="true" tabindex="-1"></a>       <span class="at">color =</span> <span class="st">&quot;Compound&quot;</span>)<span class="sc">+</span></span>
<span id="cb4-12"><a href="datanalysis.html#cb4-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_colour_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;red&quot;</span>, <span class="st">&quot;darkgoldenrod1&quot;</span>, <span class="st">&quot;green&quot;</span>, <span class="st">&quot;royalblue3&quot;</span>, <span class="st">&quot;violet&quot;</span>))<span class="sc">+</span></span>
<span id="cb4-13"><a href="datanalysis.html#cb4-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.key.size =</span> <span class="fu">unit</span>(<span class="fl">0.75</span>,<span class="st">&quot;line&quot;</span>),</span>
<span id="cb4-14"><a href="datanalysis.html#cb4-14" aria-hidden="true" tabindex="-1"></a>        <span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">8</span>))</span>
<span id="cb4-15"><a href="datanalysis.html#cb4-15" aria-hidden="true" tabindex="-1"></a>plot_chr</span></code></pre></div>
<div class="figure">
<img src="_main_files/figure-html/alphabetical_plot-1.png" alt="Number of C. elegans offspring under a number of circumstances where the x-axis is ordered alphabetically" width="672" />
<p class="caption">
(#fig:alphabetical_plot)Number of C. elegans offspring under a number of circumstances where the x-axis is ordered alphabetically
</p>
</div>
<p>The x-axis labels are ordered alphabetically because the data type of the compConcentration is character instead of double. R probably read it this way because Excel adds an E to showcase exponential values.</p>
</div>
<div id="fixing-the-characterdouble-issue" class="section level2 hasAnchor" number="4.2">
<h2><span class="header-section-number">4.2</span> Fixing the character/double issue<a href="datanalysis.html#fixing-the-characterdouble-issue" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>After changing the data type from character to double and adding jitter to the plot it looks as follows:</p>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="datanalysis.html#cb5-1" aria-hidden="true" tabindex="-1"></a><span class="co">#Change data type from chr to dbl</span></span>
<span id="cb5-2"><a href="datanalysis.html#cb5-2" aria-hidden="true" tabindex="-1"></a>celegans_data<span class="sc">$</span>compConcentration <span class="ot">&lt;-</span> <span class="fu">as.double</span>(celegans_data<span class="sc">$</span>compConcentration)</span>
<span id="cb5-3"><a href="datanalysis.html#cb5-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-4"><a href="datanalysis.html#cb5-4" aria-hidden="true" tabindex="-1"></a><span class="co">#Create a scatter plot for the concentration in nM</span></span>
<span id="cb5-5"><a href="datanalysis.html#cb5-5" aria-hidden="true" tabindex="-1"></a>celegans_data_nM <span class="ot">&lt;-</span> celegans_data <span class="sc">%&gt;%</span> <span class="fu">filter</span>(compUnit <span class="sc">==</span> <span class="st">&quot;nM&quot;</span>)</span>
<span id="cb5-6"><a href="datanalysis.html#cb5-6" aria-hidden="true" tabindex="-1"></a>plot_nM <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="at">data =</span> celegans_data_nM, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">log10</span>(compConcentration), <span class="at">y =</span> RawData))<span class="sc">+</span></span>
<span id="cb5-7"><a href="datanalysis.html#cb5-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_jitter</span>(<span class="fu">aes</span>(<span class="at">color =</span> compName, <span class="co">#add jitter</span></span>
<span id="cb5-8"><a href="datanalysis.html#cb5-8" aria-hidden="true" tabindex="-1"></a>                  <span class="at">shape =</span> expType),</span>
<span id="cb5-9"><a href="datanalysis.html#cb5-9" aria-hidden="true" tabindex="-1"></a>              <span class="at">width =</span> <span class="fl">0.5</span>, <span class="at">height =</span> <span class="fl">0.2</span>)<span class="sc">+</span></span>
<span id="cb5-10"><a href="datanalysis.html#cb5-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">&quot;log10 concentration (nM)&quot;</span>,</span>
<span id="cb5-11"><a href="datanalysis.html#cb5-11" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">&quot;Number of offspring&quot;</span>)<span class="sc">+</span></span>
<span id="cb5-12"><a href="datanalysis.html#cb5-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">coord_cartesian</span>(<span class="at">ylim =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">120</span>))<span class="sc">+</span></span>
<span id="cb5-13"><a href="datanalysis.html#cb5-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_shape_manual</span>(<span class="at">values =</span> <span class="dv">3</span>)<span class="sc">+</span></span>
<span id="cb5-14"><a href="datanalysis.html#cb5-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_colour_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;red&quot;</span>, <span class="st">&quot;darkgoldenrod1&quot;</span>, <span class="st">&quot;royalblue3&quot;</span>))<span class="sc">+</span></span>
<span id="cb5-15"><a href="datanalysis.html#cb5-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">&quot;none&quot;</span>)</span>
<span id="cb5-16"><a href="datanalysis.html#cb5-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-17"><a href="datanalysis.html#cb5-17" aria-hidden="true" tabindex="-1"></a><span class="co">#Create a scatter plot for the concentration in pct</span></span>
<span id="cb5-18"><a href="datanalysis.html#cb5-18" aria-hidden="true" tabindex="-1"></a>celegans_data_pct <span class="ot">&lt;-</span> celegans_data <span class="sc">%&gt;%</span> <span class="fu">filter</span>(compUnit <span class="sc">==</span> <span class="st">&quot;pct&quot;</span>)</span>
<span id="cb5-19"><a href="datanalysis.html#cb5-19" aria-hidden="true" tabindex="-1"></a>plot_pct <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="at">data =</span> celegans_data_pct, <span class="fu">aes</span>(<span class="at">x =</span> expType, <span class="at">y =</span> RawData))<span class="sc">+</span></span>
<span id="cb5-20"><a href="datanalysis.html#cb5-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_jitter</span>(<span class="fu">aes</span>(<span class="at">color =</span> compName, <span class="co">#add jitter</span></span>
<span id="cb5-21"><a href="datanalysis.html#cb5-21" aria-hidden="true" tabindex="-1"></a>                  <span class="at">shape =</span> expType),</span>
<span id="cb5-22"><a href="datanalysis.html#cb5-22" aria-hidden="true" tabindex="-1"></a>              <span class="at">size =</span> <span class="fl">1.3</span>,</span>
<span id="cb5-23"><a href="datanalysis.html#cb5-23" aria-hidden="true" tabindex="-1"></a>              <span class="at">width =</span> <span class="fl">0.1</span>, <span class="at">height =</span> <span class="fl">0.075</span>)<span class="sc">+</span></span>
<span id="cb5-24"><a href="datanalysis.html#cb5-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">&quot;Type&quot;</span>)<span class="sc">+</span></span>
<span id="cb5-25"><a href="datanalysis.html#cb5-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">coord_cartesian</span>(<span class="at">ylim =</span> <span class="fu">c</span>(<span class="fl">23.7</span>, <span class="dv">121</span>))<span class="sc">+</span></span>
<span id="cb5-26"><a href="datanalysis.html#cb5-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_colour_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;green&quot;</span>, <span class="st">&quot;violet&quot;</span>))<span class="sc">+</span></span>
<span id="cb5-27"><a href="datanalysis.html#cb5-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">&quot;none&quot;</span>,</span>
<span id="cb5-28"><a href="datanalysis.html#cb5-28" aria-hidden="true" tabindex="-1"></a>        <span class="at">axis.text.x=</span><span class="fu">element_text</span>(<span class="at">vjust=</span><span class="fl">0.5</span>, <span class="at">hjust=</span><span class="fl">0.5</span>, <span class="at">size =</span> <span class="fl">8.75</span>),</span>
<span id="cb5-29"><a href="datanalysis.html#cb5-29" aria-hidden="true" tabindex="-1"></a>        <span class="at">axis.text.y =</span> <span class="fu">element_blank</span>(), </span>
<span id="cb5-30"><a href="datanalysis.html#cb5-30" aria-hidden="true" tabindex="-1"></a>        <span class="at">axis.ticks.y =</span> <span class="fu">element_blank</span>(), </span>
<span id="cb5-31"><a href="datanalysis.html#cb5-31" aria-hidden="true" tabindex="-1"></a>        <span class="at">axis.title.y =</span> <span class="fu">element_blank</span>())<span class="sc">+</span> <span class="co">#remove any y-axis labeling</span></span>
<span id="cb5-32"><a href="datanalysis.html#cb5-32" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rotate_axis_labels</span>(<span class="st">&quot;x&quot;</span>, <span class="dv">90</span>)</span>
<span id="cb5-33"><a href="datanalysis.html#cb5-33" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-34"><a href="datanalysis.html#cb5-34" aria-hidden="true" tabindex="-1"></a><span class="co">#obtain legend</span></span>
<span id="cb5-35"><a href="datanalysis.html#cb5-35" aria-hidden="true" tabindex="-1"></a>legend <span class="ot">&lt;-</span> <span class="fu">get_legend</span>(plot_chr)</span>
<span id="cb5-36"><a href="datanalysis.html#cb5-36" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-37"><a href="datanalysis.html#cb5-37" aria-hidden="true" tabindex="-1"></a><span class="co">#combine figures and legend for readability</span></span>
<span id="cb5-38"><a href="datanalysis.html#cb5-38" aria-hidden="true" tabindex="-1"></a><span class="fu">ggdraw</span>(<span class="fu">plot_grid</span>(<span class="fu">plot_grid</span>(plot_nM, plot_pct),</span>
<span id="cb5-39"><a href="datanalysis.html#cb5-39" aria-hidden="true" tabindex="-1"></a>                 <span class="fu">plot_grid</span>(<span class="cn">NULL</span>, legend, <span class="at">ncol =</span> <span class="dv">3</span>),</span>
<span id="cb5-40"><a href="datanalysis.html#cb5-40" aria-hidden="true" tabindex="-1"></a>                 <span class="at">rel_widths =</span> <span class="fu">c</span>(<span class="dv">1</span>, <span class="fl">0.35</span>)))<span class="sc">+</span></span>
<span id="cb5-41"><a href="datanalysis.html#cb5-41" aria-hidden="true" tabindex="-1"></a>  <span class="fu">plot_annotation</span>(<span class="st">&quot;Number of C. elegans offspring under</span><span class="sc">\n</span><span class="st">a number of circumstances&quot;</span>)</span></code></pre></div>
<p><img src="_main_files/figure-html/unnamed-chunk-1-1.png" width="672" /></p>
<p>The positive control for this experiment is ethanol. The negative control for this experiment is S-medium.</p>
</div>
<div id="think-about-how-you-would-analyze-this-experiment-to-learn-whether-there-is-indeed-an-effect-of-different-concentrations-on-offspring-count-and-whether-the-different-compounds-have-a-different-curve-ic50.-write-down-you-analysis-as-a-step-wise-plan." class="section level2 hasAnchor" number="4.3">
<h2><span class="header-section-number">4.3</span> Think about how you would analyze this experiment to learn whether there is indeed an effect of different concentrations on offspring count and whether the different compounds have a different curve (IC50). Write down you analysis as a step-wise plan.<a href="datanalysis.html#think-about-how-you-would-analyze-this-experiment-to-learn-whether-there-is-indeed-an-effect-of-different-concentrations-on-offspring-count-and-whether-the-different-compounds-have-a-different-curve-ic50.-write-down-you-analysis-as-a-step-wise-plan." class="anchor-section" aria-label="Anchor link to header"></a></h2>
</div>
<div id="normalize-the-data-for-the-controlnegative-in-such-a-way-that-the-mean-value-for-controlnegative-is-exactly-equal-to-1-and-that-all-other-values-are-expressed-as-a-fraction-thereof.-rerun-your-graphs-with-the-normalized-data." class="section level2 hasAnchor" number="4.4">
<h2><span class="header-section-number">4.4</span> Normalize the data for the controlNegative in such a way that the mean value for controlNegative is exactly equal to 1 and that all other values are expressed as a fraction thereof. Rerun your graphs with the normalized data.<a href="datanalysis.html#normalize-the-data-for-the-controlnegative-in-such-a-way-that-the-mean-value-for-controlnegative-is-exactly-equal-to-1-and-that-all-other-values-are-expressed-as-a-fraction-thereof.-rerun-your-graphs-with-the-normalized-data." class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="datanalysis.html#cb6-1" aria-hidden="true" tabindex="-1"></a><span class="co">#Obtain the mean of the RawData (negativeControl)</span></span>
<span id="cb6-2"><a href="datanalysis.html#cb6-2" aria-hidden="true" tabindex="-1"></a>controlNeg <span class="ot">&lt;-</span> celegans_data <span class="sc">%&gt;%</span> <span class="fu">filter</span>(compName <span class="sc">==</span> <span class="st">&quot;S-medium&quot;</span>) <span class="sc">%&gt;%</span> <span class="fu">summarize</span>(<span class="at">mean =</span> <span class="fu">mean</span>(RawData, <span class="at">na.rm =</span> <span class="cn">TRUE</span>))</span>
<span id="cb6-3"><a href="datanalysis.html#cb6-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-4"><a href="datanalysis.html#cb6-4" aria-hidden="true" tabindex="-1"></a><span class="co">#Use the mean to calculate fractions</span></span>
<span id="cb6-5"><a href="datanalysis.html#cb6-5" aria-hidden="true" tabindex="-1"></a>mutated <span class="ot">&lt;-</span> celegans_data <span class="sc">%&gt;%</span> <span class="fu">filter</span>(RawData <span class="sc">&gt;</span> <span class="dv">0</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb6-6"><a href="datanalysis.html#cb6-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(RawData, compName, compConcentration, expType) <span class="sc">%&gt;%</span> <span class="fu">na.omit</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb6-7"><a href="datanalysis.html#cb6-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">normalized =</span> RawData<span class="sc">/</span>controlNeg<span class="sc">$</span>mean)</span>
<span id="cb6-8"><a href="datanalysis.html#cb6-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-9"><a href="datanalysis.html#cb6-9" aria-hidden="true" tabindex="-1"></a><span class="co">#Obtain the means of the other 2 control groups</span></span>
<span id="cb6-10"><a href="datanalysis.html#cb6-10" aria-hidden="true" tabindex="-1"></a>controlPos <span class="ot">&lt;-</span> mutated <span class="sc">%&gt;%</span> <span class="fu">filter</span>(expType <span class="sc">==</span> <span class="st">&quot;controlPositive&quot;</span>) <span class="sc">%&gt;%</span> <span class="fu">summarize</span>(<span class="at">mean =</span> <span class="fu">mean</span>(normalized, <span class="at">na.rm =</span> <span class="cn">TRUE</span>))</span>
<span id="cb6-11"><a href="datanalysis.html#cb6-11" aria-hidden="true" tabindex="-1"></a>controlVeh <span class="ot">&lt;-</span> mutated <span class="sc">%&gt;%</span> <span class="fu">filter</span>(expType <span class="sc">==</span> <span class="st">&quot;controlVehicleA&quot;</span>) <span class="sc">%&gt;%</span> <span class="fu">summarize</span>(<span class="at">mean =</span> <span class="fu">mean</span>(normalized, <span class="at">na.rm =</span> <span class="cn">TRUE</span>))</span>
<span id="cb6-12"><a href="datanalysis.html#cb6-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-13"><a href="datanalysis.html#cb6-13" aria-hidden="true" tabindex="-1"></a>mutated <span class="sc">%&gt;%</span> <span class="fu">filter</span>(compName <span class="sc">==</span> <span class="st">&quot;2,6-diisopropylnaphthalene&quot;</span> <span class="sc">|</span> compName <span class="sc">==</span> <span class="st">&quot;decane&quot;</span> <span class="sc">|</span> compName <span class="sc">==</span> <span class="st">&quot;naphthalene&quot;</span>) <span class="sc">%&gt;%</span></span>
<span id="cb6-14"><a href="datanalysis.html#cb6-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">log10</span>(compConcentration), <span class="at">y =</span> normalized))<span class="sc">+</span></span>
<span id="cb6-15"><a href="datanalysis.html#cb6-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_jitter</span>(<span class="fu">aes</span>(<span class="at">color =</span> compName), <span class="at">width =</span> <span class="fl">0.5</span>, <span class="at">height =</span> <span class="fl">0.1</span>)<span class="sc">+</span></span>
<span id="cb6-16"><a href="datanalysis.html#cb6-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_colour_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;red&quot;</span>, <span class="st">&quot;darkgoldenrod1&quot;</span>,  <span class="st">&quot;royalblue3&quot;</span>))<span class="sc">+</span></span>
<span id="cb6-17"><a href="datanalysis.html#cb6-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">coord_cartesian</span>(<span class="at">ylim =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="fl">1.5</span>))<span class="sc">+</span></span>
<span id="cb6-18"><a href="datanalysis.html#cb6-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">&quot;log10 concentration (nM)&quot;</span>,</span>
<span id="cb6-19"><a href="datanalysis.html#cb6-19" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">&quot;Normalized number of offspring&quot;</span>,</span>
<span id="cb6-20"><a href="datanalysis.html#cb6-20" aria-hidden="true" tabindex="-1"></a>       <span class="at">title =</span> <span class="st">&quot;Number of C. elegans offspring as a fraction</span><span class="sc">\n</span><span class="st">of the negative control group&quot;</span>,</span>
<span id="cb6-21"><a href="datanalysis.html#cb6-21" aria-hidden="true" tabindex="-1"></a>       <span class="at">color =</span> <span class="st">&quot;Compound&quot;</span>)<span class="sc">+</span></span>
<span id="cb6-22"><a href="datanalysis.html#cb6-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> <span class="dv">1</span>, <span class="at">color =</span> <span class="st">&quot;violet&quot;</span>)<span class="sc">+</span></span>
<span id="cb6-23"><a href="datanalysis.html#cb6-23" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> controlPos<span class="sc">$</span>mean, <span class="at">color =</span> <span class="st">&quot;green&quot;</span>)<span class="sc">+</span></span>
<span id="cb6-24"><a href="datanalysis.html#cb6-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> controlVeh<span class="sc">$</span>mean, <span class="at">color =</span> <span class="st">&quot;green&quot;</span>, <span class="at">linetype =</span> <span class="st">&quot;longdash&quot;</span>)<span class="sc">+</span></span>
<span id="cb6-25"><a href="datanalysis.html#cb6-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">guides</span>(<span class="at">color =</span> <span class="st">&quot;none&quot;</span>)<span class="sc">+</span></span>
<span id="cb6-26"><a href="datanalysis.html#cb6-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="st">&quot;text&quot;</span>, <span class="at">x =</span> <span class="sc">-</span><span class="fl">3.8</span>, <span class="at">y =</span> <span class="fl">0.55</span>, <span class="at">label =</span> <span class="st">&quot;controlPositive&quot;</span>, <span class="at">size =</span> <span class="dv">2</span>)<span class="sc">+</span></span>
<span id="cb6-27"><a href="datanalysis.html#cb6-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="st">&quot;text&quot;</span>, <span class="at">x =</span> <span class="sc">-</span><span class="fl">3.8</span>, <span class="at">y =</span> <span class="fl">0.95</span>, <span class="at">label =</span> <span class="st">&quot;controlNegative&quot;</span>, <span class="at">size =</span> <span class="dv">2</span>)<span class="sc">+</span></span>
<span id="cb6-28"><a href="datanalysis.html#cb6-28" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="st">&quot;text&quot;</span>, <span class="at">x =</span> <span class="sc">-</span><span class="fl">3.8</span>, <span class="at">y =</span> <span class="fl">1.05</span>, <span class="at">label =</span> <span class="st">&quot;controlVehicleA&quot;</span>, <span class="at">size =</span> <span class="dv">2</span>)<span class="sc">+</span></span>
<span id="cb6-29"><a href="datanalysis.html#cb6-29" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_wrap</span>(<span class="sc">~</span> compName)</span></code></pre></div>
<p><img src="_main_files/figure-html/J-1.png" width="672" /></p>
<p>This way it’s easier to read the graph. Everything below the pink line means less offspring than control C. elegans and everything above it means more.</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="guerillaframework.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="primates.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
