<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>3 Analyzing and visualizing data | portfolio_book.knit</title>
  <meta name="description" content="" />
  <meta name="generator" content="bookdown 0.31 and GitBook 2.6.7" />

  <meta property="og:title" content="3 Analyzing and visualizing data | portfolio_book.knit" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="3 Analyzing and visualizing data | portfolio_book.knit" />
  
  
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="reproducibleresearch.html"/>
<link rel="next" href="drugscreening.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<script src="libs/htmlwidgets-1.6.1/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.11.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.11.1/plotly-latest.min.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="booktheme.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<div class="titleandname">Portfolio Nine Luijendijk</div>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Curriculum Vitae</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#contact"><i class="fa fa-check"></i>Contact</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#education"><i class="fa fa-check"></i>Education</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#experience"><i class="fa fa-check"></i>Experience</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#skills"><i class="fa fa-check"></i>Skills</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#languages"><i class="fa fa-check"></i>Languages</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="guerillaframework.html"><a href="guerillaframework.html"><i class="fa fa-check"></i><b>1</b> Guerilla Analytics framework</a></li>
<li class="chapter" data-level="2" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html"><i class="fa fa-check"></i><b>2</b> Reproducible Research</a>
<ul>
<li class="chapter" data-level="2.1" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#scoring-the-reproducibility-of-a-research-article"><i class="fa fa-check"></i><b>2.1</b> Scoring the reproducibility of a research article</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#reproducible-research-and-criteria"><i class="fa fa-check"></i><b>2.1.1</b> Reproducible research and criteria</a></li>
<li class="chapter" data-level="2.1.2" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#the-research-article"><i class="fa fa-check"></i><b>2.1.2</b> The research article</a></li>
<li class="chapter" data-level="2.1.3" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#reproducibility-of-the-research"><i class="fa fa-check"></i><b>2.1.3</b> Reproducibility of the research</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#scoring-the-reproducibility-of-the-code-of-the-research"><i class="fa fa-check"></i><b>2.2</b> Scoring the reproducibility of the code of the research</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#criteria-for-the-code"><i class="fa fa-check"></i><b>2.2.1</b> Criteria for the code</a></li>
<li class="chapter" data-level="2.2.2" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#the-code"><i class="fa fa-check"></i><b>2.2.2</b> The code</a></li>
<li class="chapter" data-level="" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#install-andor-load-packages"><i class="fa fa-check"></i>Install and/or load packages</a></li>
<li class="chapter" data-level="" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#data"><i class="fa fa-check"></i>Data</a></li>
<li class="chapter" data-level="" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#multiple-item-psychological-measures"><i class="fa fa-check"></i>Multiple-item Psychological Measures</a></li>
<li class="chapter" data-level="" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#install-andor-load-packages-1"><i class="fa fa-check"></i>Install and/or load packages</a></li>
<li class="chapter" data-level="" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#data-1"><i class="fa fa-check"></i>Data</a></li>
<li class="chapter" data-level="" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#multiple-item-psychological-measures-1"><i class="fa fa-check"></i>Multiple-item Psychological Measures</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#final-rating"><i class="fa fa-check"></i><b>2.3</b> Final rating</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="datanalysis.html"><a href="datanalysis.html"><i class="fa fa-check"></i><b>3</b> Analyzing and visualizing data</a>
<ul>
<li class="chapter" data-level="3.1" data-path="datanalysis.html"><a href="datanalysis.html#preparing-the-data"><i class="fa fa-check"></i><b>3.1</b> Preparing the data</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="datanalysis.html"><a href="datanalysis.html#inspecting-the-data-in-r"><i class="fa fa-check"></i><b>3.1.1</b> Inspecting the data in R</a></li>
<li class="chapter" data-level="3.1.2" data-path="datanalysis.html"><a href="datanalysis.html#fixing-the-characterdouble-issue"><i class="fa fa-check"></i><b>3.1.2</b> Fixing the character/double issue</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="datanalysis.html"><a href="datanalysis.html#analyzing-and-visualizing-the-data"><i class="fa fa-check"></i><b>3.2</b> Analyzing and visualizing the data</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="datanalysis.html"><a href="datanalysis.html#plan-for-ic50ed50-visualization"><i class="fa fa-check"></i><b>3.2.1</b> Plan for IC50/ED50 visualization</a></li>
<li class="chapter" data-level="3.2.2" data-path="datanalysis.html"><a href="datanalysis.html#normalizing-the-data"><i class="fa fa-check"></i><b>3.2.2</b> Normalizing the data</a></li>
<li class="chapter" data-level="3.2.3" data-path="datanalysis.html"><a href="datanalysis.html#creating-a-legend"><i class="fa fa-check"></i><b>3.2.3</b> Creating a legend</a></li>
<li class="chapter" data-level="3.2.4" data-path="datanalysis.html"><a href="datanalysis.html#generating-the-normalized-plot"><i class="fa fa-check"></i><b>3.2.4</b> Generating the normalized plot</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="drugscreening.html"><a href="drugscreening.html"><i class="fa fa-check"></i><b>4</b> Drug Screening</a></li>
<li class="chapter" data-level="5" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html"><i class="fa fa-check"></i><b>5</b> Working with relational databases</a>
<ul>
<li class="chapter" data-level="5.1" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#preparing-the-data-1"><i class="fa fa-check"></i><b>5.1</b> Preparing the data</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#introduction"><i class="fa fa-check"></i><b>5.1.1</b> Introduction</a></li>
<li class="chapter" data-level="5.1.2" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#importing-the-data-and-making-it-tidy"><i class="fa fa-check"></i><b>5.1.2</b> Importing the data and making it tidy</a></li>
<li class="chapter" data-level="5.1.3" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#connecting-to-the-database"><i class="fa fa-check"></i><b>5.1.3</b> Connecting to the database</a></li>
<li class="chapter" data-level="5.1.4" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#exporting-the-tidy-dataframes-to-the-database"><i class="fa fa-check"></i><b>5.1.4</b> Exporting the tidy dataframes to the database</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#inspecting-the-data"><i class="fa fa-check"></i><b>5.2</b> Inspecting the data</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#inspecting-the-data-using-sql"><i class="fa fa-check"></i><b>5.2.1</b> Inspecting the data using SQL</a></li>
<li class="chapter" data-level="5.2.2" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#inspecting-the-data-using-r"><i class="fa fa-check"></i><b>5.2.2</b> Inspecting the data using R</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#combining-the-dataframes"><i class="fa fa-check"></i><b>5.3</b> Combining the dataframes</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#preparing-the-tidy-dataframes"><i class="fa fa-check"></i><b>5.3.1</b> Preparing the tidy dataframes</a></li>
<li class="chapter" data-level="5.3.2" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#joining-the-dataframes-together"><i class="fa fa-check"></i><b>5.3.2</b> Joining the dataframes together</a></li>
<li class="chapter" data-level="5.3.3" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#loading-the-combined-tables-into-r"><i class="fa fa-check"></i><b>5.3.3</b> Loading the combined tables into R</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#visualizing-the-data"><i class="fa fa-check"></i><b>5.4</b> Visualizing the data</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#normalizing-the-data-1"><i class="fa fa-check"></i><b>5.4.1</b> Normalizing the data</a></li>
<li class="chapter" data-level="5.4.2" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#calculating-descriptive-statistics"><i class="fa fa-check"></i><b>5.4.2</b> Calculating descriptive statistics</a></li>
<li class="chapter" data-level="5.4.3" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#visualizing-correlation"><i class="fa fa-check"></i><b>5.4.3</b> Visualizing correlation</a></li>
<li class="chapter" data-level="5.4.4" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#visualizing-the-dengue-data"><i class="fa fa-check"></i><b>5.4.4</b> Visualizing the dengue data</a></li>
<li class="chapter" data-level="5.4.5" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#visualizing-the-flu-data"><i class="fa fa-check"></i><b>5.4.5</b> Visualizing the flu data</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="parameterized.html"><a href="parameterized.html"><i class="fa fa-check"></i><b>6</b> Parameterized report on COVID-19</a>
<ul>
<li class="chapter" data-level="6.1" data-path="parameterized.html"><a href="parameterized.html#parameterized-reports"><i class="fa fa-check"></i><b>6.1</b> Parameterized reports</a></li>
<li class="chapter" data-level="6.2" data-path="parameterized.html"><a href="parameterized.html#the-parameterized-covid-19-report"><i class="fa fa-check"></i><b>6.2</b> The parameterized COVID-19 report</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="parameterized.html"><a href="parameterized.html#the-parameters"><i class="fa fa-check"></i><b>6.2.1</b> The parameters</a></li>
<li class="chapter" data-level="6.2.2" data-path="parameterized.html"><a href="parameterized.html#rendering-the-report"><i class="fa fa-check"></i><b>6.2.2</b> Rendering the report</a></li>
<li class="chapter" data-level="6.2.3" data-path="parameterized.html"><a href="parameterized.html#using-different-parameters"><i class="fa fa-check"></i><b>6.2.3</b> Using different parameters</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="vignette.html"><a href="vignette.html"><i class="fa fa-check"></i><b>7</b> R package vignette</a>
<ul>
<li class="chapter" data-level="7.1" data-path="vignette.html"><a href="vignette.html#creating-the-package"><i class="fa fa-check"></i><b>7.1</b> Creating the package</a></li>
<li class="chapter" data-level="7.2" data-path="vignette.html"><a href="vignette.html#installing-and-loading-the-package"><i class="fa fa-check"></i><b>7.2</b> Installing and loading the package</a></li>
<li class="chapter" data-level="7.3" data-path="vignette.html"><a href="vignette.html#package-contents"><i class="fa fa-check"></i><b>7.3</b> Package contents</a>
<ul>
<li class="chapter" data-level="7.3.1" data-path="vignette.html"><a href="vignette.html#dataset-measlesdata"><i class="fa fa-check"></i><b>7.3.1</b> Dataset measlesdata</a></li>
<li class="chapter" data-level="7.3.2" data-path="vignette.html"><a href="vignette.html#function-datasummarizer"><i class="fa fa-check"></i><b>7.3.2</b> Function dataSummarizer()</a></li>
<li class="chapter" data-level="7.3.3" data-path="vignette.html"><a href="vignette.html#function-illnessplot"><i class="fa fa-check"></i><b>7.3.3</b> Function illnessPlot()</a></li>
<li class="chapter" data-level="7.3.4" data-path="vignette.html"><a href="vignette.html#function-fastaimporter.r"><i class="fa fa-check"></i><b>7.3.4</b> Function fastaImporter.R</a></li>
<li class="chapter" data-level="7.3.5" data-path="vignette.html"><a href="vignette.html#function-gotermanalysis"><i class="fa fa-check"></i><b>7.3.5</b> Function gotermAnalysis()</a></li>
<li class="chapter" data-level="7.3.6" data-path="vignette.html"><a href="vignette.html#function-gotermplot"><i class="fa fa-check"></i><b>7.3.6</b> Function gotermPlot()</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="vignette.html"><a href="vignette.html#additional-files"><i class="fa fa-check"></i><b>7.4</b> Additional files</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="primates.html"><a href="primates.html"><i class="fa fa-check"></i><b>8</b> Generating a phylogenetic tree in R</a>
<ul>
<li class="chapter" data-level="8.1" data-path="primates.html"><a href="primates.html#the-plan"><i class="fa fa-check"></i><b>8.1</b> The plan</a></li>
<li class="chapter" data-level="8.2" data-path="primates.html"><a href="primates.html#preparatory-research"><i class="fa fa-check"></i><b>8.2</b> Preparatory research</a></li>
<li class="chapter" data-level="8.3" data-path="primates.html"><a href="primates.html#getting-started"><i class="fa fa-check"></i><b>8.3</b> Getting started</a></li>
<li class="chapter" data-level="8.4" data-path="primates.html"><a href="primates.html#generating-the-first-tree"><i class="fa fa-check"></i><b>8.4</b> Generating the first tree</a>
<ul>
<li class="chapter" data-level="8.4.1" data-path="primates.html"><a href="primates.html#obtaining-fasta-sequences"><i class="fa fa-check"></i><b>8.4.1</b> Obtaining FASTA sequences</a></li>
<li class="chapter" data-level="8.4.2" data-path="primates.html"><a href="primates.html#performing-multiple-sequence-alignment"><i class="fa fa-check"></i><b>8.4.2</b> Performing multiple sequence alignment</a></li>
<li class="chapter" data-level="8.4.3" data-path="primates.html"><a href="primates.html#importing-the-aligned-dna"><i class="fa fa-check"></i><b>8.4.3</b> Importing the aligned dna</a></li>
<li class="chapter" data-level="8.4.4" data-path="primates.html"><a href="primates.html#calculating-the-genetic-distances-between-the-sequences-and-drawing-a-tree"><i class="fa fa-check"></i><b>8.4.4</b> Calculating the genetic distances between the sequences and drawing a tree</a></li>
</ul></li>
<li class="chapter" data-level="8.5" data-path="primates.html"><a href="primates.html#different-types-of-phylogenetic-models"><i class="fa fa-check"></i><b>8.5</b> Different types of phylogenetic models</a>
<ul>
<li class="chapter" data-level="8.5.1" data-path="primates.html"><a href="primates.html#finding-the-best-model"><i class="fa fa-check"></i><b>8.5.1</b> Finding the best model</a></li>
<li class="chapter" data-level="8.5.2" data-path="primates.html"><a href="primates.html#generating-a-maximum-likelihood-tree"><i class="fa fa-check"></i><b>8.5.2</b> Generating a maximum likelihood tree</a></li>
<li class="chapter" data-level="8.5.3" data-path="primates.html"><a href="primates.html#visualizing-the-maximum-likelihood-cladogram"><i class="fa fa-check"></i><b>8.5.3</b> Visualizing the maximum likelihood cladogram</a></li>
<li class="chapter" data-level="8.5.4" data-path="primates.html"><a href="primates.html#rooting-the-tree"><i class="fa fa-check"></i><b>8.5.4</b> Rooting the tree</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="datanalysis" class="section level1" number="3">
<h1><span class="header-section-number">3</span> Analyzing and visualizing data</h1>
<hr />
<p>The data that is to be visualized comes from an experiment involving C. elegans nematodes and was supplied by J. Louter of the INT/ILC. In this experiment, the number of offspring of adult C. elegans was counted after exposing them to different substances in multiple concentrations.</p>
<div id="preparing-the-data" class="section level2" number="3.1">
<h2><span class="header-section-number">3.1</span> Preparing the data</h2>
<div id="inspecting-the-data-in-r" class="section level3" number="3.1.1">
<h3><span class="header-section-number">3.1.1</span> Inspecting the data in R</h3>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb15-1"><a href="datanalysis.html#cb15-1" aria-hidden="true" tabindex="-1"></a>celegans_data <span class="ot">&lt;-</span> <span class="fu">read_excel</span>(<span class="fu">here</span>(<span class="st">&quot;data_raw/CE.LIQ.FLOW.062_Tidydata.xlsx&quot;</span>))</span></code></pre></div>
<p>Before the data can be visualized, it has to be cleaned up. The dataframe is already in tidy format, but the data types of the columns need to be changed. Expected data types would be double/integer for RawData (number of offspring), character for compName (name of the compound) and double for compConcentration (concentration of the compound). The actual data types of the columns are double for RawData, character for compName and character for compConcentration. The data type for compConcentration has not been correctly assigned which will cause the following issue if it’s not changed:</p>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb16-1"><a href="datanalysis.html#cb16-1" aria-hidden="true" tabindex="-1"></a><span class="co">#Create a scatter plot</span></span>
<span id="cb16-2"><a href="datanalysis.html#cb16-2" aria-hidden="true" tabindex="-1"></a>plot_chr <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="at">data =</span> celegans_data, <span class="fu">aes</span>(<span class="at">x =</span> compConcentration, <span class="at">y =</span> RawData))<span class="sc">+</span></span>
<span id="cb16-3"><a href="datanalysis.html#cb16-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="fu">aes</span>(<span class="at">color =</span> compName,</span>
<span id="cb16-4"><a href="datanalysis.html#cb16-4" aria-hidden="true" tabindex="-1"></a>                 <span class="at">shape =</span> expType),</span>
<span id="cb16-5"><a href="datanalysis.html#cb16-5" aria-hidden="true" tabindex="-1"></a>             <span class="at">size =</span> <span class="dv">1</span>)<span class="sc">+</span></span>
<span id="cb16-6"><a href="datanalysis.html#cb16-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">&quot;Concentration&quot;</span>,</span>
<span id="cb16-7"><a href="datanalysis.html#cb16-7" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">&quot;Number of offspring&quot;</span>,</span>
<span id="cb16-8"><a href="datanalysis.html#cb16-8" aria-hidden="true" tabindex="-1"></a>       <span class="at">title =</span> <span class="st">&quot;Number of C. elegans offspring under</span><span class="sc">\n</span><span class="st">a number of circumstances, alphabetical&quot;</span>,</span>
<span id="cb16-9"><a href="datanalysis.html#cb16-9" aria-hidden="true" tabindex="-1"></a>       <span class="at">shape =</span> <span class="st">&quot;Type&quot;</span>,</span>
<span id="cb16-10"><a href="datanalysis.html#cb16-10" aria-hidden="true" tabindex="-1"></a>       <span class="at">color =</span> <span class="st">&quot;Compound&quot;</span>)<span class="sc">+</span></span>
<span id="cb16-11"><a href="datanalysis.html#cb16-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_colour_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;red&quot;</span>, <span class="st">&quot;darkgoldenrod1&quot;</span>, <span class="st">&quot;green&quot;</span>, <span class="st">&quot;royalblue3&quot;</span>, <span class="st">&quot;violet&quot;</span>))<span class="sc">+</span></span>
<span id="cb16-12"><a href="datanalysis.html#cb16-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>()<span class="sc">+</span></span>
<span id="cb16-13"><a href="datanalysis.html#cb16-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.key.size =</span> <span class="fu">unit</span>(<span class="fl">0.75</span>,<span class="st">&quot;line&quot;</span>),</span>
<span id="cb16-14"><a href="datanalysis.html#cb16-14" aria-hidden="true" tabindex="-1"></a>        <span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">8</span>),</span>
<span id="cb16-15"><a href="datanalysis.html#cb16-15" aria-hidden="true" tabindex="-1"></a>        <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>, <span class="at">vjust =</span> <span class="fl">0.5</span>, <span class="at">hjust =</span> <span class="dv">1</span>))</span>
<span id="cb16-16"><a href="datanalysis.html#cb16-16" aria-hidden="true" tabindex="-1"></a>plot_chr</span></code></pre></div>
<div class="figure"><span style="display:block;" id="fig:alphabeticalplot"></span>
<img src="portfolio_book_files/figure-html/alphabeticalplot-1.png" alt="Number of C. elegans offspring under a number of circumstances where the x-axis is ordered alphabetically" width="672" />
<p class="caption">
Figure 3.1: Number of C. elegans offspring under a number of circumstances where the x-axis is ordered alphabetically
</p>
</div>
<p>The x-axis labels are ordered alphabetically because the data type of the compConcentration is character instead of double. R probably read it this way because Excel adds an E to showcase exponential values.</p>
</div>
<div id="fixing-the-characterdouble-issue" class="section level3" number="3.1.2">
<h3><span class="header-section-number">3.1.2</span> Fixing the character/double issue</h3>
<p>Since not all compounds share the same compUnit (the unit in which their concentration is measured), 2 separate plots need to be made and then combined: one for the compounds measured in nM and one for the compounds measured in percentage. After changing the data type from character to double and adding jitter to the plot to make it more readable it looks as follows:</p>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb17-1"><a href="datanalysis.html#cb17-1" aria-hidden="true" tabindex="-1"></a><span class="co">#Change data type from chr to dbl</span></span>
<span id="cb17-2"><a href="datanalysis.html#cb17-2" aria-hidden="true" tabindex="-1"></a>celegans_data<span class="sc">$</span>compConcentration <span class="ot">&lt;-</span> <span class="fu">as.double</span>(celegans_data<span class="sc">$</span>compConcentration)</span>
<span id="cb17-3"><a href="datanalysis.html#cb17-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-4"><a href="datanalysis.html#cb17-4" aria-hidden="true" tabindex="-1"></a><span class="co">#Create a scatter plot for the concentration in nM</span></span>
<span id="cb17-5"><a href="datanalysis.html#cb17-5" aria-hidden="true" tabindex="-1"></a>celegans_data_nM <span class="ot">&lt;-</span> celegans_data <span class="sc">%&gt;%</span> <span class="fu">filter</span>(compUnit <span class="sc">==</span> <span class="st">&quot;nM&quot;</span>)</span>
<span id="cb17-6"><a href="datanalysis.html#cb17-6" aria-hidden="true" tabindex="-1"></a>plot_nM <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="at">data =</span> celegans_data_nM, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">log10</span>(compConcentration), <span class="at">y =</span> RawData))<span class="sc">+</span></span>
<span id="cb17-7"><a href="datanalysis.html#cb17-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_jitter</span>(<span class="fu">aes</span>(<span class="at">color =</span> compName, <span class="co">#add jitter</span></span>
<span id="cb17-8"><a href="datanalysis.html#cb17-8" aria-hidden="true" tabindex="-1"></a>                  <span class="at">shape =</span> expType),</span>
<span id="cb17-9"><a href="datanalysis.html#cb17-9" aria-hidden="true" tabindex="-1"></a>              <span class="at">width =</span> <span class="fl">0.5</span>, <span class="at">height =</span> <span class="fl">0.2</span>)<span class="sc">+</span></span>
<span id="cb17-10"><a href="datanalysis.html#cb17-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">&quot;log10 concentration (nM)&quot;</span>,</span>
<span id="cb17-11"><a href="datanalysis.html#cb17-11" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">&quot;Number of offspring&quot;</span>)<span class="sc">+</span></span>
<span id="cb17-12"><a href="datanalysis.html#cb17-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">coord_cartesian</span>(<span class="at">ylim =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">120</span>))<span class="sc">+</span></span>
<span id="cb17-13"><a href="datanalysis.html#cb17-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_shape_manual</span>(<span class="at">values =</span> <span class="dv">3</span>)<span class="sc">+</span></span>
<span id="cb17-14"><a href="datanalysis.html#cb17-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_colour_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;red&quot;</span>, <span class="st">&quot;darkgoldenrod1&quot;</span>, <span class="st">&quot;royalblue3&quot;</span>))<span class="sc">+</span></span>
<span id="cb17-15"><a href="datanalysis.html#cb17-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>()<span class="sc">+</span></span>
<span id="cb17-16"><a href="datanalysis.html#cb17-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">&quot;none&quot;</span>)</span>
<span id="cb17-17"><a href="datanalysis.html#cb17-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-18"><a href="datanalysis.html#cb17-18" aria-hidden="true" tabindex="-1"></a><span class="co">#Create a scatter plot for the concentration in pct</span></span>
<span id="cb17-19"><a href="datanalysis.html#cb17-19" aria-hidden="true" tabindex="-1"></a>celegans_data_pct <span class="ot">&lt;-</span> celegans_data <span class="sc">%&gt;%</span> <span class="fu">filter</span>(compUnit <span class="sc">==</span> <span class="st">&quot;pct&quot;</span>)</span>
<span id="cb17-20"><a href="datanalysis.html#cb17-20" aria-hidden="true" tabindex="-1"></a>plot_pct <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="at">data =</span> celegans_data_pct, <span class="fu">aes</span>(<span class="at">x =</span> expType, <span class="at">y =</span> RawData))<span class="sc">+</span></span>
<span id="cb17-21"><a href="datanalysis.html#cb17-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_jitter</span>(<span class="fu">aes</span>(<span class="at">color =</span> compName, <span class="co">#add jitter</span></span>
<span id="cb17-22"><a href="datanalysis.html#cb17-22" aria-hidden="true" tabindex="-1"></a>                  <span class="at">shape =</span> expType),</span>
<span id="cb17-23"><a href="datanalysis.html#cb17-23" aria-hidden="true" tabindex="-1"></a>              <span class="at">size =</span> <span class="fl">1.3</span>,</span>
<span id="cb17-24"><a href="datanalysis.html#cb17-24" aria-hidden="true" tabindex="-1"></a>              <span class="at">width =</span> <span class="fl">0.1</span>, <span class="at">height =</span> <span class="fl">0.075</span>)<span class="sc">+</span></span>
<span id="cb17-25"><a href="datanalysis.html#cb17-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">&quot;Type&quot;</span>)<span class="sc">+</span></span>
<span id="cb17-26"><a href="datanalysis.html#cb17-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">coord_cartesian</span>(<span class="at">ylim =</span> <span class="fu">c</span>(<span class="fl">23.7</span>, <span class="dv">121</span>))<span class="sc">+</span></span>
<span id="cb17-27"><a href="datanalysis.html#cb17-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_colour_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;green&quot;</span>, <span class="st">&quot;violet&quot;</span>))<span class="sc">+</span></span>
<span id="cb17-28"><a href="datanalysis.html#cb17-28" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>()<span class="sc">+</span></span>
<span id="cb17-29"><a href="datanalysis.html#cb17-29" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">&quot;none&quot;</span>,</span>
<span id="cb17-30"><a href="datanalysis.html#cb17-30" aria-hidden="true" tabindex="-1"></a>        <span class="at">axis.text.x=</span><span class="fu">element_text</span>(<span class="at">vjust=</span><span class="fl">0.5</span>, <span class="at">hjust=</span><span class="fl">0.5</span>, <span class="at">size =</span> <span class="fl">8.75</span>),</span>
<span id="cb17-31"><a href="datanalysis.html#cb17-31" aria-hidden="true" tabindex="-1"></a>        <span class="at">axis.text.y =</span> <span class="fu">element_blank</span>(), </span>
<span id="cb17-32"><a href="datanalysis.html#cb17-32" aria-hidden="true" tabindex="-1"></a>        <span class="at">axis.ticks.y =</span> <span class="fu">element_blank</span>(), </span>
<span id="cb17-33"><a href="datanalysis.html#cb17-33" aria-hidden="true" tabindex="-1"></a>        <span class="at">axis.title.y =</span> <span class="fu">element_blank</span>())<span class="sc">+</span> <span class="co">#remove any y-axis labeling</span></span>
<span id="cb17-34"><a href="datanalysis.html#cb17-34" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rotate_axis_labels</span>(<span class="st">&quot;x&quot;</span>, <span class="dv">90</span>)</span>
<span id="cb17-35"><a href="datanalysis.html#cb17-35" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-36"><a href="datanalysis.html#cb17-36" aria-hidden="true" tabindex="-1"></a><span class="co">#obtain legend</span></span>
<span id="cb17-37"><a href="datanalysis.html#cb17-37" aria-hidden="true" tabindex="-1"></a>legend <span class="ot">&lt;-</span> <span class="fu">get_legend</span>(plot_chr)</span>
<span id="cb17-38"><a href="datanalysis.html#cb17-38" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-39"><a href="datanalysis.html#cb17-39" aria-hidden="true" tabindex="-1"></a><span class="co">#combine figures and legend for readability</span></span>
<span id="cb17-40"><a href="datanalysis.html#cb17-40" aria-hidden="true" tabindex="-1"></a><span class="fu">ggdraw</span>(<span class="fu">plot_grid</span>(<span class="fu">plot_grid</span>(plot_nM, plot_pct),</span>
<span id="cb17-41"><a href="datanalysis.html#cb17-41" aria-hidden="true" tabindex="-1"></a>                 <span class="fu">plot_grid</span>(<span class="cn">NULL</span>, legend, <span class="at">ncol =</span> <span class="dv">3</span>),</span>
<span id="cb17-42"><a href="datanalysis.html#cb17-42" aria-hidden="true" tabindex="-1"></a>                 <span class="at">rel_widths =</span> <span class="fu">c</span>(<span class="dv">1</span>, <span class="fl">0.35</span>)))<span class="sc">+</span></span>
<span id="cb17-43"><a href="datanalysis.html#cb17-43" aria-hidden="true" tabindex="-1"></a>  <span class="fu">plot_annotation</span>(<span class="st">&quot;Number of C. elegans offspring under</span><span class="sc">\n</span><span class="st">a number of circumstances&quot;</span>)</span></code></pre></div>
<div class="figure"><span style="display:block;" id="fig:scatterplotcelegans"></span>
<img src="portfolio_book_files/figure-html/scatterplotcelegans-1.png" alt="Total number of C. elegans offspring under a number of circumstances, with on the y-axis the number of offspring and on the x-axis the log10 concentration in nM." width="672" />
<p class="caption">
Figure 3.2: Total number of C. elegans offspring under a number of circumstances, with on the y-axis the number of offspring and on the x-axis the log10 concentration in nM.
</p>
</div>
<p>The positive control for this experiment is ethanol. The negative control for this experiment is S-medium.</p>
</div>
</div>
<div id="analyzing-and-visualizing-the-data" class="section level2" number="3.2">
<h2><span class="header-section-number">3.2</span> Analyzing and visualizing the data</h2>
<div id="plan-for-ic50ed50-visualization" class="section level3" number="3.2.1">
<h3><span class="header-section-number">3.2.1</span> Plan for IC50/ED50 visualization</h3>
<p>Since this chapter is focused on visualization, there won’t be a statistical analysis of the dose-response interactions. If I were to perform a statistical analysis, it would be done using the {drc} package <span class="citation">(<a href="#ref-ritzDoseResponseAnalysisUsing2015" role="doc-biblioref">Ritz et al. 2015</a>)</span>, which has functions to calculate IC50s and ED50s, which then could be compared. Using {ggplot2} <span class="citation">(<a href="#ref-wickhamGgplot2ElegantGraphics2016" role="doc-biblioref">Hadley Wickham 2016</a>)</span>, a scatter plot can be made with on the y-axis the number of offspring and on the x-axis the compound concentration. The calculated IC50/ED50 values could be added as lines over the plot, giving every compound (line) a different color. An ANOVA can also be conducted to look for differences between concentrations, after checking if the data comes from a normal distribution using the Shapiro-Wilk test.</p>
</div>
<div id="normalizing-the-data" class="section level3" number="3.2.2">
<h3><span class="header-section-number">3.2.2</span> Normalizing the data</h3>
<p>To make is easier to read the results visualized in the plot, the number of offspring was normalized as a fraction of the negative control, meaning any value under 1 shows a reduction in the number of offspring. To normalize the data, the following code was used:</p>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="datanalysis.html#cb18-1" aria-hidden="true" tabindex="-1"></a><span class="co">#Obtain the mean of the RawData (negativeControl)</span></span>
<span id="cb18-2"><a href="datanalysis.html#cb18-2" aria-hidden="true" tabindex="-1"></a>controlNeg <span class="ot">&lt;-</span> celegans_data <span class="sc">%&gt;%</span> <span class="fu">filter</span>(compName <span class="sc">==</span> <span class="st">&quot;S-medium&quot;</span>) <span class="sc">%&gt;%</span> <span class="fu">summarize</span>(<span class="at">mean =</span> <span class="fu">mean</span>(RawData, <span class="at">na.rm =</span> <span class="cn">TRUE</span>))</span>
<span id="cb18-3"><a href="datanalysis.html#cb18-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb18-4"><a href="datanalysis.html#cb18-4" aria-hidden="true" tabindex="-1"></a><span class="co">#Use the mean to calculate fractions</span></span>
<span id="cb18-5"><a href="datanalysis.html#cb18-5" aria-hidden="true" tabindex="-1"></a>mutated <span class="ot">&lt;-</span> celegans_data <span class="sc">%&gt;%</span> <span class="fu">filter</span>(RawData <span class="sc">&gt;</span> <span class="dv">0</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb18-6"><a href="datanalysis.html#cb18-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(RawData, compName, compConcentration, expType) <span class="sc">%&gt;%</span> <span class="fu">na.omit</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb18-7"><a href="datanalysis.html#cb18-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">normalized =</span> RawData<span class="sc">/</span>controlNeg<span class="sc">$</span>mean)</span>
<span id="cb18-8"><a href="datanalysis.html#cb18-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb18-9"><a href="datanalysis.html#cb18-9" aria-hidden="true" tabindex="-1"></a><span class="co">#Obtain the means of the other 2 control groups</span></span>
<span id="cb18-10"><a href="datanalysis.html#cb18-10" aria-hidden="true" tabindex="-1"></a>controlPos <span class="ot">&lt;-</span> mutated <span class="sc">%&gt;%</span> <span class="fu">filter</span>(expType <span class="sc">==</span> <span class="st">&quot;controlPositive&quot;</span>) <span class="sc">%&gt;%</span> <span class="fu">summarize</span>(<span class="at">mean =</span> <span class="fu">mean</span>(normalized, <span class="at">na.rm =</span> <span class="cn">TRUE</span>))</span>
<span id="cb18-11"><a href="datanalysis.html#cb18-11" aria-hidden="true" tabindex="-1"></a>controlVeh <span class="ot">&lt;-</span> mutated <span class="sc">%&gt;%</span> <span class="fu">filter</span>(expType <span class="sc">==</span> <span class="st">&quot;controlVehicleA&quot;</span>) <span class="sc">%&gt;%</span> <span class="fu">summarize</span>(<span class="at">mean =</span> <span class="fu">mean</span>(normalized, <span class="at">na.rm =</span> <span class="cn">TRUE</span>))</span></code></pre></div>
</div>
<div id="creating-a-legend" class="section level3" number="3.2.3">
<h3><span class="header-section-number">3.2.3</span> Creating a legend</h3>
<p>To give the final figure a proper legend, the legend needs to be generated in a separate figure since we don’t want to include the test-compounds in it. The following code was used to generate a legend, making use of dummy data.</p>
<details>
<summary>
Generating a legend using dummy data
</summary>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb19-1"><a href="datanalysis.html#cb19-1" aria-hidden="true" tabindex="-1"></a>dummydf <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="at">a =</span> <span class="fu">rep</span>(<span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">2</span>, <span class="dv">3</span>), <span class="at">each=</span><span class="dv">2</span>), <span class="co">#create dummy dataframe for the dummy plot</span></span>
<span id="cb19-2"><a href="datanalysis.html#cb19-2" aria-hidden="true" tabindex="-1"></a>                      <span class="at">b =</span> <span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">4</span>, <span class="dv">6</span>, <span class="dv">3</span>, <span class="dv">4</span>, <span class="dv">7</span>),</span>
<span id="cb19-3"><a href="datanalysis.html#cb19-3" aria-hidden="true" tabindex="-1"></a>                      <span class="at">Control =</span> <span class="fu">rep</span>(<span class="fu">c</span>(<span class="st">&quot;controlNegative&quot;</span>, <span class="st">&quot;controlPositive&quot;</span>, <span class="st">&quot;controlVehicleA&quot;</span>), <span class="at">each=</span><span class="dv">2</span>))</span>
<span id="cb19-4"><a href="datanalysis.html#cb19-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-5"><a href="datanalysis.html#cb19-5" aria-hidden="true" tabindex="-1"></a>dummyplot <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(dummydf, <span class="fu">aes</span>(<span class="at">x=</span>a, <span class="at">y=</span>b, <span class="at">group=</span>Control))<span class="sc">+</span> <span class="co">#create dummy plot to obtain its legend</span></span>
<span id="cb19-6"><a href="datanalysis.html#cb19-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>(<span class="fu">aes</span>(<span class="at">linetype=</span>Control, <span class="at">color=</span>Control))<span class="sc">+</span></span>
<span id="cb19-7"><a href="datanalysis.html#cb19-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_linetype_manual</span>(<span class="at">values=</span><span class="fu">c</span>(<span class="st">&quot;solid&quot;</span>, <span class="st">&quot;solid&quot;</span>, <span class="st">&quot;longdash&quot;</span>))<span class="sc">+</span></span>
<span id="cb19-8"><a href="datanalysis.html#cb19-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values=</span><span class="fu">c</span>(<span class="st">&quot;violet&quot;</span>, <span class="st">&quot;green&quot;</span>, <span class="st">&quot;green&quot;</span>))</span>
<span id="cb19-9"><a href="datanalysis.html#cb19-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-10"><a href="datanalysis.html#cb19-10" aria-hidden="true" tabindex="-1"></a>dummylegend <span class="ot">&lt;-</span> <span class="fu">get_legend</span>(dummyplot) <span class="co">#obtain dummy legend</span></span></code></pre></div>
</details>
</div>
<div id="generating-the-normalized-plot" class="section level3" number="3.2.4">
<h3><span class="header-section-number">3.2.4</span> Generating the normalized plot</h3>
<p>With the data normalized and the legend obtained, the normalized plot can be generated.</p>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb20-1"><a href="datanalysis.html#cb20-1" aria-hidden="true" tabindex="-1"></a><span class="co">#create normalized plot</span></span>
<span id="cb20-2"><a href="datanalysis.html#cb20-2" aria-hidden="true" tabindex="-1"></a>plotfraction <span class="ot">&lt;-</span> mutated <span class="sc">%&gt;%</span> <span class="fu">filter</span>(compName <span class="sc">==</span> <span class="st">&quot;2,6-diisopropylnaphthalene&quot;</span> <span class="sc">|</span> compName <span class="sc">==</span> <span class="st">&quot;decane&quot;</span> <span class="sc">|</span> compName <span class="sc">==</span> <span class="st">&quot;naphthalene&quot;</span>) <span class="sc">%&gt;%</span></span>
<span id="cb20-3"><a href="datanalysis.html#cb20-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">log10</span>(compConcentration), <span class="at">y =</span> normalized))<span class="sc">+</span></span>
<span id="cb20-4"><a href="datanalysis.html#cb20-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_jitter</span>(<span class="fu">aes</span>(<span class="at">color =</span> compName), <span class="at">width =</span> <span class="fl">0.5</span>, <span class="at">height =</span> <span class="fl">0.1</span>)<span class="sc">+</span></span>
<span id="cb20-5"><a href="datanalysis.html#cb20-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_colour_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;red&quot;</span>, <span class="st">&quot;darkgoldenrod1&quot;</span>,  <span class="st">&quot;royalblue3&quot;</span>))<span class="sc">+</span></span>
<span id="cb20-6"><a href="datanalysis.html#cb20-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">coord_cartesian</span>(<span class="at">ylim =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="fl">1.5</span>))<span class="sc">+</span></span>
<span id="cb20-7"><a href="datanalysis.html#cb20-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">&quot;log10 concentration (nM)&quot;</span>,</span>
<span id="cb20-8"><a href="datanalysis.html#cb20-8" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">&quot;Normalized number of offspring&quot;</span>,</span>
<span id="cb20-9"><a href="datanalysis.html#cb20-9" aria-hidden="true" tabindex="-1"></a>       <span class="at">title =</span> <span class="st">&quot;Number of C. elegans offspring as a fraction</span><span class="sc">\n</span><span class="st">of the negative control group&quot;</span>,</span>
<span id="cb20-10"><a href="datanalysis.html#cb20-10" aria-hidden="true" tabindex="-1"></a>       <span class="at">color =</span> <span class="st">&quot;Compound&quot;</span>)<span class="sc">+</span></span>
<span id="cb20-11"><a href="datanalysis.html#cb20-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> <span class="dv">1</span>, <span class="at">color =</span> <span class="st">&quot;violet&quot;</span>)<span class="sc">+</span></span>
<span id="cb20-12"><a href="datanalysis.html#cb20-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> controlPos<span class="sc">$</span>mean, <span class="at">color =</span> <span class="st">&quot;green&quot;</span>)<span class="sc">+</span></span>
<span id="cb20-13"><a href="datanalysis.html#cb20-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> controlVeh<span class="sc">$</span>mean, <span class="at">color =</span> <span class="st">&quot;green&quot;</span>, <span class="at">linetype =</span> <span class="st">&quot;longdash&quot;</span>)<span class="sc">+</span></span>
<span id="cb20-14"><a href="datanalysis.html#cb20-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>()<span class="sc">+</span></span>
<span id="cb20-15"><a href="datanalysis.html#cb20-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">guides</span>(<span class="at">color =</span> <span class="st">&quot;none&quot;</span>)<span class="sc">+</span></span>
<span id="cb20-16"><a href="datanalysis.html#cb20-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_wrap</span>(<span class="sc">~</span> compName)</span>
<span id="cb20-17"><a href="datanalysis.html#cb20-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-18"><a href="datanalysis.html#cb20-18" aria-hidden="true" tabindex="-1"></a><span class="co">#combine figure and legend</span></span>
<span id="cb20-19"><a href="datanalysis.html#cb20-19" aria-hidden="true" tabindex="-1"></a><span class="fu">ggdraw</span>(<span class="fu">plot_grid</span>(plotfraction, dummylegend,</span>
<span id="cb20-20"><a href="datanalysis.html#cb20-20" aria-hidden="true" tabindex="-1"></a>       <span class="at">rel_widths =</span> <span class="fu">c</span>(<span class="dv">5</span>, <span class="dv">1</span>)))</span></code></pre></div>
<div class="figure"><span style="display:block;" id="fig:normalizedplot"></span>
<img src="portfolio_book_files/figure-html/normalizedplot-1.png" alt="Number of C. elegans offspring as a fraction of the negative control group, with on the y-axis the normalized number of offspring and on the x-axis the log10 concentration of the compounds in nM." width="672" />
<p class="caption">
Figure 3.3: Number of C. elegans offspring as a fraction of the negative control group, with on the y-axis the normalized number of offspring and on the x-axis the log10 concentration of the compounds in nM.
</p>
</div>
<p>This way it’s easier to read the graph. Everything below the pink line means less offspring than control C. elegans and everything above it means more. The figure (<a href="datanalysis.html#fig:normalizedplot">3.3</a>) shows a decrease in the number of offspring for all 3 groups.</p>

</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-ritzDoseResponseAnalysisUsing2015" class="csl-entry">
Ritz, Christian, Florent Baty, Jens C. Streibig, and Daniel Gerhard. 2015. <span>“Dose-Response Analysis Using R.”</span> <em>PLOS ONE</em> 10 (12): e0146021. <a href="https://doi.org/10.1371/journal.pone.0146021">https://doi.org/10.1371/journal.pone.0146021</a>.
</div>
<div id="ref-wickhamGgplot2ElegantGraphics2016" class="csl-entry">
Wickham, Hadley. 2016. <span>“Ggplot2: Elegant Graphics for Data Analysis.”</span> 2016. <a href="https://ggplot2.tidyverse.org">https://ggplot2.tidyverse.org</a>.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="reproducibleresearch.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="drugscreening.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
}
});
});
</script>

</body>

</html>
