<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>8 Generating a phylogenetic tree in R | portfolio_book.knit</title>
  <meta name="description" content="" />
  <meta name="generator" content="bookdown 0.30 and GitBook 2.6.7" />

  <meta property="og:title" content="8 Generating a phylogenetic tree in R | portfolio_book.knit" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="8 Generating a phylogenetic tree in R | portfolio_book.knit" />
  
  
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="rpackage.html"/>
<link rel="next" href="references.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.11.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.11.1/plotly-latest.min.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Curriculum Vitae</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#contact"><i class="fa fa-check"></i>Contact</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#education"><i class="fa fa-check"></i>Education</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#experience"><i class="fa fa-check"></i>Experience</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#skills"><i class="fa fa-check"></i>Skills</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#languages"><i class="fa fa-check"></i>Languages</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="guerillaframework.html"><a href="guerillaframework.html"><i class="fa fa-check"></i><b>1</b> Guerilla Analytics framework</a></li>
<li class="chapter" data-level="2" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html"><i class="fa fa-check"></i><b>2</b> Reproducible Research</a></li>
<li class="chapter" data-level="3" data-path="datanalysis.html"><a href="datanalysis.html"><i class="fa fa-check"></i><b>3</b> Analyzing and visualizing data</a>
<ul>
<li class="chapter" data-level="3.1" data-path="datanalysis.html"><a href="datanalysis.html#preparing-the-data"><i class="fa fa-check"></i><b>3.1</b> Preparing the data</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="datanalysis.html"><a href="datanalysis.html#inspecting-the-data-in-r"><i class="fa fa-check"></i><b>3.1.1</b> Inspecting the data in R</a></li>
<li class="chapter" data-level="3.1.2" data-path="datanalysis.html"><a href="datanalysis.html#fixing-the-characterdouble-issue"><i class="fa fa-check"></i><b>3.1.2</b> Fixing the character/double issue</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="datanalysis.html"><a href="datanalysis.html#analyzing-and-visualizing-the-data"><i class="fa fa-check"></i><b>3.2</b> Analyzing and visualizing the data</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="datanalysis.html"><a href="datanalysis.html#plan-for-ic50ed50-visualization"><i class="fa fa-check"></i><b>3.2.1</b> Plan for IC50/ED50 visualization</a></li>
<li class="chapter" data-level="3.2.2" data-path="datanalysis.html"><a href="datanalysis.html#normalizing-the-data"><i class="fa fa-check"></i><b>3.2.2</b> Normalizing the data</a></li>
<li class="chapter" data-level="3.2.3" data-path="datanalysis.html"><a href="datanalysis.html#creating-a-legend"><i class="fa fa-check"></i><b>3.2.3</b> Creating a legend</a></li>
<li class="chapter" data-level="3.2.4" data-path="datanalysis.html"><a href="datanalysis.html#generating-the-normalized-plot"><i class="fa fa-check"></i><b>3.2.4</b> Generating the normalized plot</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="drugscreening.html"><a href="drugscreening.html"><i class="fa fa-check"></i><b>4</b> Drug Screening</a></li>
<li class="chapter" data-level="5" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html"><i class="fa fa-check"></i><b>5</b> Working with relational databases</a>
<ul>
<li class="chapter" data-level="5.1" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#preparing-the-data-1"><i class="fa fa-check"></i><b>5.1</b> Preparing the data</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#introduction"><i class="fa fa-check"></i><b>5.1.1</b> Introduction</a></li>
<li class="chapter" data-level="5.1.2" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#importing-the-data-and-making-it-tidy"><i class="fa fa-check"></i><b>5.1.2</b> Importing the data and making it tidy</a></li>
<li class="chapter" data-level="5.1.3" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#connecting-to-the-database"><i class="fa fa-check"></i><b>5.1.3</b> Connecting to the database</a></li>
<li class="chapter" data-level="5.1.4" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#exporting-the-tidy-dataframes-to-the-database"><i class="fa fa-check"></i><b>5.1.4</b> Exporting the tidy dataframes to the database</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#inspecting-the-data"><i class="fa fa-check"></i><b>5.2</b> Inspecting the data</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#inspecting-the-data-using-sql"><i class="fa fa-check"></i><b>5.2.1</b> Inspecting the data using SQL</a></li>
<li class="chapter" data-level="5.2.2" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#inspecting-the-data-using-r"><i class="fa fa-check"></i><b>5.2.2</b> Inspecting the data using R</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#combining-the-dataframes"><i class="fa fa-check"></i><b>5.3</b> Combining the dataframes</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#preparing-the-tidy-dataframes"><i class="fa fa-check"></i><b>5.3.1</b> Preparing the tidy dataframes</a></li>
<li class="chapter" data-level="5.3.2" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#joining-the-dataframes-together"><i class="fa fa-check"></i><b>5.3.2</b> Joining the dataframes together</a></li>
<li class="chapter" data-level="5.3.3" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#loading-the-combined-tables-into-r"><i class="fa fa-check"></i><b>5.3.3</b> Loading the combined tables into R</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#visualizing-the-data"><i class="fa fa-check"></i><b>5.4</b> Visualizing the data</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#normalizing-the-data-1"><i class="fa fa-check"></i><b>5.4.1</b> Normalizing the data</a></li>
<li class="chapter" data-level="5.4.2" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#calculating-descriptive-statistics"><i class="fa fa-check"></i><b>5.4.2</b> Calculating descriptive statistics</a></li>
<li class="chapter" data-level="5.4.3" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#visualizing-correlation"><i class="fa fa-check"></i><b>5.4.3</b> Visualizing correlation</a></li>
<li class="chapter" data-level="5.4.4" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#visualizing-the-dengue-data"><i class="fa fa-check"></i><b>5.4.4</b> Visualizing the dengue data</a></li>
<li class="chapter" data-level="5.4.5" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#visualizing-the-flu-data"><i class="fa fa-check"></i><b>5.4.5</b> Visualizing the flu data</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="parameterizedcovid.html"><a href="parameterizedcovid.html"><i class="fa fa-check"></i><b>6</b> Parameterized report COVID-19</a></li>
<li class="chapter" data-level="7" data-path="rpackage.html"><a href="rpackage.html"><i class="fa fa-check"></i><b>7</b> Personal R package</a></li>
<li class="chapter" data-level="8" data-path="primates.html"><a href="primates.html"><i class="fa fa-check"></i><b>8</b> Generating a phylogenetic tree in R</a>
<ul>
<li class="chapter" data-level="8.1" data-path="primates.html"><a href="primates.html#the-plan"><i class="fa fa-check"></i><b>8.1</b> The plan</a></li>
<li class="chapter" data-level="8.2" data-path="primates.html"><a href="primates.html#preparatory-research"><i class="fa fa-check"></i><b>8.2</b> Preparatory research</a></li>
<li class="chapter" data-level="8.3" data-path="primates.html"><a href="primates.html#getting-started"><i class="fa fa-check"></i><b>8.3</b> Getting started</a></li>
<li class="chapter" data-level="8.4" data-path="primates.html"><a href="primates.html#generating-the-first-tree"><i class="fa fa-check"></i><b>8.4</b> Generating the first tree</a>
<ul>
<li class="chapter" data-level="8.4.1" data-path="primates.html"><a href="primates.html#obtaining-fasta-sequences"><i class="fa fa-check"></i><b>8.4.1</b> Obtaining FASTA sequences</a></li>
<li class="chapter" data-level="8.4.2" data-path="primates.html"><a href="primates.html#performing-multiple-sequence-alignment"><i class="fa fa-check"></i><b>8.4.2</b> Performing multiple sequence alignment</a></li>
<li class="chapter" data-level="8.4.3" data-path="primates.html"><a href="primates.html#importing-the-aligned-dna"><i class="fa fa-check"></i><b>8.4.3</b> Importing the aligned dna</a></li>
<li class="chapter" data-level="8.4.4" data-path="primates.html"><a href="primates.html#calculating-the-genetic-distances-between-the-sequences-and-drawing-a-tree"><i class="fa fa-check"></i><b>8.4.4</b> Calculating the genetic distances between the sequences and drawing a tree</a></li>
</ul></li>
<li class="chapter" data-level="8.5" data-path="primates.html"><a href="primates.html#different-types-of-phylogenetic-models"><i class="fa fa-check"></i><b>8.5</b> Different types of phylogenetic models</a>
<ul>
<li class="chapter" data-level="8.5.1" data-path="primates.html"><a href="primates.html#finding-the-best-model"><i class="fa fa-check"></i><b>8.5.1</b> Finding the best model</a></li>
<li class="chapter" data-level="8.5.2" data-path="primates.html"><a href="primates.html#generating-a-maximum-likelihood-tree"><i class="fa fa-check"></i><b>8.5.2</b> Generating a maximum likelihood tree</a></li>
<li class="chapter" data-level="8.5.3" data-path="primates.html"><a href="primates.html#visualizing-the-maximum-likelihood-cladogram"><i class="fa fa-check"></i><b>8.5.3</b> Visualizing the maximum likelihood cladogram</a></li>
<li class="chapter" data-level="8.5.4" data-path="primates.html"><a href="primates.html#rooting-the-tree"><i class="fa fa-check"></i><b>8.5.4</b> Rooting the tree</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="primates" class="section level1" number="8">
<h1><span class="header-section-number">8</span> Generating a phylogenetic tree in R</h1>
<div id="the-plan" class="section level2" number="8.1">
<h2><span class="header-section-number">8.1</span> The plan</h2>
<p>In 2 years’ time I want to be doing my master’s in Biology at Universiteit Leiden. I would like to specialize in Evolutionary Biology. Next semester I will be attending classes at Radboud Universiteit that are part of their minor Biology - Adaptive Organisms to get a head start on learning about evolutionary biology, both because it’s something I enjoy and to make the transition from life sciences to the master’s program easier</p>
<p>I would like to learn more about the differences between multiple primates and generate a phylogenetic tree. Prior to me starting on the actual code for this phylogenetic tree, I had to look into the different types of trees and how to generate them.</p>
</div>
<div id="preparatory-research" class="section level2" number="8.2">
<h2><span class="header-section-number">8.2</span> Preparatory research</h2>
<p>I started by looking for R packages having to do with phylogenetic research. There were ample packages to help generate phylogenetic trees, but none of them seemed to have all the tools needed. Wanting to keep my research as reproducible as possible, my main priority was to keep every single part of the research in R, which is free and open source software <span class="citation">(<a href="#ref-rcoreteamLanguageEnvironmentStatistical2014" role="doc-biblioref">R Core Team 2014</a>)</span>. To get an idea of how to get started, I read a couple of papers by researchers who’ve created phylogenetic trees. In these papers, most of the software used to generate the trees (and all the steps that come before) was not R <span class="citation">(<a href="#ref-pozziPrimatePhylogeneticRelationships2014" role="doc-biblioref">Pozzi et al. 2014</a>)</span>; <span class="citation">(<a href="#ref-perelmanMolecularPhylogenyLiving2011" role="doc-biblioref">Perelman et al. 2011</a>)</span>. Some of the programs were point and click as well <span class="citation">(<a href="#ref-suchardBayesianPhylogeneticPhylodynamic2018" role="doc-biblioref">Suchard et al. 2018</a>)</span>; <span class="citation">(<a href="#ref-vaidyaSequenceMatrixConcatenationSoftware2011" role="doc-biblioref">Vaidya, Lohman, and Meier 2011</a>)</span>, making the research less reproducible.</p>
<p>After looking at some different R packages I decided {ape} <span class="citation">(<a href="#ref-paradisApeEnvironmentModern2019" role="doc-biblioref">Paradis and Schliep 2019</a>)</span> was the best option, seeing as most packages for phylogenetic research are dependent on it <span class="citation">(<a href="#ref-jombartAdegenet31New2011" role="doc-biblioref">Jombart and Ahmed 2011</a>)</span>.
The first step in the analysis was to obtain the data from an online database. NCBI has their own command-line tool called NCBI Datasets <span class="citation">(<a href="#ref-ncbistaffNowAvailableUpdated2022" role="doc-biblioref">NCBI Staff 2022</a>)</span>. The command-line tool was installed using Miniconda3 <span class="citation">(<a href="#ref-anacondaMinicondaCondaDocumentation" role="doc-biblioref">Anaconda n.d.</a>)</span>. The tool did work fine, but it wasn’t exactly what I was looking for. This was also when it was decided to perform the analysis on the mitochondrial genomes of the primates, instead of the nuclear genomes, due to computational limitations.</p>
</div>
<div id="getting-started" class="section level2" number="8.3">
<h2><span class="header-section-number">8.3</span> Getting started</h2>
<p>I ended up making a list of the species of interest, containing their GenBank <span class="citation">(<a href="#ref-bensonGenBank2013" role="doc-biblioref">Benson et al. 2013</a>)</span> identifier, scientific name and their common name, and saving it in a text file called species.txt. This is the only part of the code that would need to be changed in case a species is to be added.</p>
<p>To keep the code tidy the packages {tidyverse} <span class="citation">(<a href="#ref-wickhamWelcomeTidyverse2019" role="doc-biblioref">Hadley Wickham et al. 2019</a>)</span> and {here} <span class="citation">(<a href="#ref-mullerHere" role="doc-biblioref">Müller n.d.</a>)</span> are used. {BiocManager} <span class="citation">(<a href="#ref-ramosInstallingBiocManager" role="doc-biblioref">Ramos and Morgan n.d.</a>)</span> was also used to install packages.</p>
</div>
<div id="generating-the-first-tree" class="section level2" number="8.4">
<h2><span class="header-section-number">8.4</span> Generating the first tree</h2>
<div id="obtaining-fasta-sequences" class="section level3" number="8.4.1">
<h3><span class="header-section-number">8.4.1</span> Obtaining FASTA sequences</h3>
<p>The file is read by R and filtered so that the comments (species’ names) are removed, and only the GenBank identifier remains. Then, using the package {rentrez} <span class="citation">(<a href="#ref-winterRentrezPackageNCBI2017" role="doc-biblioref">Winter 2017</a>)</span>, the FASTA sequences are obtained and stored in a vector. The FASTA sequences are then stored in a file so they can be used as input for the {ape} function <code>read.FASTA</code>, where the sequences will be converted to a vector of the class DNAbin.</p>
<div class="sourceCode" id="cb57"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb57-1"><a href="primates.html#cb57-1" aria-hidden="true" tabindex="-1"></a>species <span class="ot">&lt;-</span><span class="fu">scan</span>(<span class="at">file =</span> <span class="fu">here</span>(<span class="st">&quot;data/species.txt&quot;</span>), <span class="at">what =</span> <span class="st">&quot;character&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;</span><span class="sc">\n</span><span class="st">&quot;</span>, <span class="at">comment.char =</span> <span class="st">&quot;#&quot;</span>) <span class="co">#import list of species to be analyzed</span></span>
<span id="cb57-2"><a href="primates.html#cb57-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb57-3"><a href="primates.html#cb57-3" aria-hidden="true" tabindex="-1"></a>fasta_seqs <span class="ot">&lt;-</span> <span class="fu">entrez_fetch</span>(<span class="at">db =</span> <span class="st">&quot;nucleotide&quot;</span>,</span>
<span id="cb57-4"><a href="primates.html#cb57-4" aria-hidden="true" tabindex="-1"></a>             <span class="at">id =</span> species,</span>
<span id="cb57-5"><a href="primates.html#cb57-5" aria-hidden="true" tabindex="-1"></a>             <span class="at">rettype =</span> <span class="st">&quot;fasta&quot;</span>) <span class="co">#retrieve the fasta sequences of the mitochondrial genome of every species in the list</span></span>
<span id="cb57-6"><a href="primates.html#cb57-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb57-7"><a href="primates.html#cb57-7" aria-hidden="true" tabindex="-1"></a><span class="fu">write</span>(fasta_seqs, </span>
<span id="cb57-8"><a href="primates.html#cb57-8" aria-hidden="true" tabindex="-1"></a>      <span class="at">file=</span><span class="st">&quot;temp&quot;</span>)</span>
<span id="cb57-9"><a href="primates.html#cb57-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb57-10"><a href="primates.html#cb57-10" aria-hidden="true" tabindex="-1"></a><span class="fu">grep</span>(<span class="st">&quot;^*$&quot;</span>, <span class="fu">readLines</span>(<span class="st">&quot;temp&quot;</span>), <span class="at">invert =</span> <span class="cn">TRUE</span>, <span class="at">value =</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span> <span class="fu">write</span>(<span class="at">file =</span> <span class="fu">here</span>(<span class="st">&quot;data/sequences.fasta&quot;</span>)) <span class="co">#remove empty lines so read.FASTA works</span></span>
<span id="cb57-11"><a href="primates.html#cb57-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb57-12"><a href="primates.html#cb57-12" aria-hidden="true" tabindex="-1"></a><span class="fu">file.remove</span>(<span class="st">&quot;temp&quot;</span>)</span>
<span id="cb57-13"><a href="primates.html#cb57-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb57-14"><a href="primates.html#cb57-14" aria-hidden="true" tabindex="-1"></a>dna <span class="ot">&lt;-</span> <span class="fu">read.FASTA</span>(<span class="fu">here</span>(<span class="st">&quot;data/sequences.fasta&quot;</span>), <span class="at">type =</span> <span class="st">&quot;DNA&quot;</span>) <span class="co">#store the sequences in a vector of the type &quot;DNAbin&quot;</span></span></code></pre></div>
</div>
<div id="performing-multiple-sequence-alignment" class="section level3" number="8.4.2">
<h3><span class="header-section-number">8.4.2</span> Performing multiple sequence alignment</h3>
<p>Now that the dna has been stored as DNAbin, a multiple sequence alignment will be performed using Clustal Omega 1.2.3 <span class="citation">(<a href="#ref-sieversFastScalableGeneration2011" role="doc-biblioref">Sievers et al. 2011</a>)</span>. The aligned DNA is saved as a file, so the alignment only has to be performed once (since it takes a while).</p>
<div class="sourceCode" id="cb58"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb58-1"><a href="primates.html#cb58-1" aria-hidden="true" tabindex="-1"></a>dna_aligned <span class="ot">&lt;-</span> <span class="fu">clustalomega</span>(dna, <span class="at">exec =</span> <span class="fu">here</span>(<span class="st">&quot;clustalo&quot;</span>)) <span class="co">#use Clustal Omega 1.2.3 to perform multiple sequence alignment</span></span>
<span id="cb58-2"><a href="primates.html#cb58-2" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(dna_aligned, <span class="fu">here</span>(<span class="st">&quot;data/dna_aligned.RDS&quot;</span>)) <span class="co">#save as file so the code only has to run once</span></span></code></pre></div>
</div>
<div id="importing-the-aligned-dna" class="section level3" number="8.4.3">
<h3><span class="header-section-number">8.4.3</span> Importing the aligned dna</h3>
<div class="sourceCode" id="cb59"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb59-1"><a href="primates.html#cb59-1" aria-hidden="true" tabindex="-1"></a>dna_aligned <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="fu">here</span>(<span class="st">&quot;data/dna_aligned.RDS&quot;</span>)) <span class="co">#import the aligned dna</span></span></code></pre></div>
</div>
<div id="calculating-the-genetic-distances-between-the-sequences-and-drawing-a-tree" class="section level3" number="8.4.4">
<h3><span class="header-section-number">8.4.4</span> Calculating the genetic distances between the sequences and drawing a tree</h3>
<p>The different dna sequences have been aligned and now the genetic distances between the sequences can be calculated using the Tamura and Nei 1993 model (TN93) <span class="citation">(<a href="#ref-tamuraEstimationNumberNucleotide1993" role="doc-biblioref">Tamura and Nei 1993</a>)</span>. The distance matrix can be used as input to calculate the phylogenetic tree. The used algorithm is the BIONJ algorithm of Gascuel <span class="citation">(<a href="#ref-gascuelBIONJImprovedVersion1997" role="doc-biblioref">Gascuel 1997</a>)</span>.
The generated tree is visualized as a cladogram using the package {ggtree} <span class="citation">(<a href="#ref-yuGgtreePackageVisualization2017" role="doc-biblioref">Yu et al. 2017</a>)</span>. It’s visualized as a cladogram for now to keep it easy to read while it’s being improved.</p>
<div class="sourceCode" id="cb60"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb60-1"><a href="primates.html#cb60-1" aria-hidden="true" tabindex="-1"></a>dna_dist <span class="ot">&lt;-</span> <span class="fu">dist.dna</span>(dna_aligned, <span class="at">model =</span> <span class="st">&quot;TN93&quot;</span>) <span class="co">#calculate genetic distances and store in a vector of the type &quot;dist&quot;</span></span>
<span id="cb60-2"><a href="primates.html#cb60-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb60-3"><a href="primates.html#cb60-3" aria-hidden="true" tabindex="-1"></a>treebionj <span class="ot">&lt;-</span> <span class="fu">bionj</span>(dna_dist) <span class="co">#create a tree of the class &quot;phylo&quot; using the method neighbor joining</span></span>
<span id="cb60-4"><a href="primates.html#cb60-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb60-5"><a href="primates.html#cb60-5" aria-hidden="true" tabindex="-1"></a><span class="fu">ggtree</span>(treebionj, <span class="at">branch.length =</span> <span class="st">&quot;none&quot;</span>, <span class="at">ladderize =</span> F)<span class="sc">+</span> <span class="co">#visualize the tree as a cladogram</span></span>
<span id="cb60-6"><a href="primates.html#cb60-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_tree</span>()<span class="sc">+</span></span>
<span id="cb60-7"><a href="primates.html#cb60-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_tiplab</span>(<span class="at">size =</span> <span class="dv">6</span>, <span class="at">as_ylab =</span> T)<span class="sc">+</span></span>
<span id="cb60-8"><a href="primates.html#cb60-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">&quot;Unrooted neighbor joining cladogram of</span><span class="sc">\n</span><span class="st">mitochondrial genomes of 15 primates and the golden hamster&quot;</span>)</span></code></pre></div>
<p><img src="portfolio_book_files/figure-html/Calculate%20the%20genetic%20distances%20between%20the%20sequences-1.png" width="672" /></p>
</div>
</div>
<div id="different-types-of-phylogenetic-models" class="section level2" number="8.5">
<h2><span class="header-section-number">8.5</span> Different types of phylogenetic models</h2>
<p>This first tree was generated using an improved neighbor joining model, which is just one of the four most commonly used <span class="citation">(<a href="#ref-yoshidaEfficienciesNJpMaximum2016" role="doc-biblioref">Yoshida and Nei 2016</a>)</span>. Neighbor joining only looks at the distance between the sequences, but not at smaller details in the sequences <span class="citation">(<a href="#ref-yoshidaEfficienciesNJpMaximum2016" role="doc-biblioref">Yoshida and Nei 2016</a>)</span>. A different method is maximum likelihood, where the likelihood of the different possible states of the tree is calculated, and the tree with the highest likelihood is generated <span class="citation">(<a href="#ref-dharMaximumLikelihoodPhylogenetic2016" role="doc-biblioref">Dhar and Minin 2016</a>)</span>. Two other popular methods are Bayesian inference and maximum parsimony <span class="citation">(<a href="#ref-dharMaximumLikelihoodPhylogenetic2016" role="doc-biblioref">Dhar and Minin 2016</a>)</span>; <span class="citation">(<a href="#ref-yoshidaEfficienciesNJpMaximum2016" role="doc-biblioref">Yoshida and Nei 2016</a>)</span>. Since maximum likelihood is the preferred method in publications <span class="citation">(<a href="#ref-dharMaximumLikelihoodPhylogenetic2016" role="doc-biblioref">Dhar and Minin 2016</a>)</span>, that is the method I will be using.</p>
<div id="finding-the-best-model" class="section level3" number="8.5.1">
<h3><span class="header-section-number">8.5.1</span> Finding the best model</h3>
<p>The input for the {phangorn} <code>modelTest</code> function is an object of the class phyDat, so first the DNAbin object has to be converted:</p>
<div class="sourceCode" id="cb61"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb61-1"><a href="primates.html#cb61-1" aria-hidden="true" tabindex="-1"></a>dna_phydat <span class="ot">&lt;-</span> <span class="fu">as.phyDat</span>(dna_aligned) <span class="co">#convert object to class phyDat</span></span></code></pre></div>
<p>The model test is ran and the model with the lowest Akaike information criterion (AIC, calculated by the {phangorn} function <code>modelTest</code>), making it the best model <span class="citation">(<a href="#ref-cavanaughAkaikeInformationCriterion2019" role="doc-biblioref">Cavanaugh and Neath 2019</a>)</span>, is saved, again so this only has to be performed once as it takes quite long.</p>
<div class="sourceCode" id="cb62"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb62-1"><a href="primates.html#cb62-1" aria-hidden="true" tabindex="-1"></a>modeltest <span class="ot">&lt;-</span> <span class="fu">modelTest</span>(dna_phydat) <span class="co">#run the model test</span></span>
<span id="cb62-2"><a href="primates.html#cb62-2" aria-hidden="true" tabindex="-1"></a>optmodel <span class="ot">&lt;-</span> modeltest<span class="sc">$</span>Model[modeltest<span class="sc">$</span>AIC<span class="sc">==</span><span class="fu">min</span>(modeltest<span class="sc">$</span>AIC)] <span class="co">#find the model with the lowest AIC</span></span>
<span id="cb62-3"><a href="primates.html#cb62-3" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(optmodel, <span class="fu">here</span>(<span class="st">&quot;data/optmodel&quot;</span>)) <span class="co">#save as file so the code only has to run once</span></span></code></pre></div>
<div class="sourceCode" id="cb63"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb63-1"><a href="primates.html#cb63-1" aria-hidden="true" tabindex="-1"></a>optmodel <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="fu">here</span>(<span class="st">&quot;data/optmodel&quot;</span>)) <span class="co">#import the data again</span></span></code></pre></div>
</div>
<div id="generating-a-maximum-likelihood-tree" class="section level3" number="8.5.2">
<h3><span class="header-section-number">8.5.2</span> Generating a maximum likelihood tree</h3>
<p>Using the neighbor joining tree, the aligned dna and the best model the maximum likelihood tree can be computed:</p>
<div class="sourceCode" id="cb64"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb64-1"><a href="primates.html#cb64-1" aria-hidden="true" tabindex="-1"></a>mlparsed <span class="ot">&lt;-</span> <span class="fu">pml</span>(treebionj, dna_phydat) <span class="co">#generate the maximum likelihood tree</span></span></code></pre></div>
<p>The <code>optim.pml</code> function from {phangorn} is used to optimize the different model parameters. The model found using <code>modelTest</code> is not an option in the <code>optim.pml</code> function, so the closest option is used.</p>
<div class="sourceCode" id="cb65"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb65-1"><a href="primates.html#cb65-1" aria-hidden="true" tabindex="-1"></a>optmodel <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">&quot;</span><span class="sc">\\</span><span class="st">+.*&quot;</span>,<span class="st">&quot;&quot;</span>, optmodel) <span class="co">#change the string so the optim.pml function can read it</span></span>
<span id="cb65-2"><a href="primates.html#cb65-2" aria-hidden="true" tabindex="-1"></a>mlparsedoptim <span class="ot">&lt;-</span> <span class="fu">optim.pml</span>(mlparsed, <span class="at">optNni=</span><span class="cn">TRUE</span>, <span class="at">model =</span> optmodel) <span class="co">#optimize the tree</span></span></code></pre></div>
</div>
<div id="visualizing-the-maximum-likelihood-cladogram" class="section level3" number="8.5.3">
<h3><span class="header-section-number">8.5.3</span> Visualizing the maximum likelihood cladogram</h3>
<div class="sourceCode" id="cb66"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb66-1"><a href="primates.html#cb66-1" aria-hidden="true" tabindex="-1"></a>treeml <span class="ot">&lt;-</span> <span class="fu">ladderize</span>(mlparsedoptim<span class="sc">$</span>tree) <span class="co">#ladderize the tree</span></span>
<span id="cb66-2"><a href="primates.html#cb66-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb66-3"><a href="primates.html#cb66-3" aria-hidden="true" tabindex="-1"></a><span class="fu">ggtree</span>(treeml, <span class="at">branch.length =</span> <span class="st">&quot;none&quot;</span>, <span class="at">ladderize =</span> F)<span class="sc">+</span> <span class="co">#visualize the tree as a cladogram</span></span>
<span id="cb66-4"><a href="primates.html#cb66-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_tree</span>()<span class="sc">+</span></span>
<span id="cb66-5"><a href="primates.html#cb66-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_tiplab</span>(<span class="at">size =</span> <span class="dv">6</span>, <span class="at">as_ylab =</span> T)<span class="sc">+</span></span>
<span id="cb66-6"><a href="primates.html#cb66-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">&quot;Unrooted maximum likelihood cladogram of</span><span class="sc">\n</span><span class="st">mitochondrial genomes of 15 primates and the golden hamster&quot;</span>)</span></code></pre></div>
<p><img src="portfolio_book_files/figure-html/Visualize%20unrooted%20maximum%20likelihood%20tree-1.png" width="672" /></p>
</div>
<div id="rooting-the-tree" class="section level3" number="8.5.4">
<h3><span class="header-section-number">8.5.4</span> Rooting the tree</h3>
<p>The last step is to root the tree and clean up the tip labels (species names). It is important to root the tree, trees that aren’t rooted correctly may be misleading <span class="citation">(<a href="#ref-kineneRootingTreesMethods2016" role="doc-biblioref">Kinene et al. 2016</a>)</span>. There are multiple ways to root a tree, I will be using an “outgroup”, a species that is genetically very different from the other species in the tree <span class="citation">(<a href="#ref-kineneRootingTreesMethods2016" role="doc-biblioref">Kinene et al. 2016</a>)</span>.</p>
<div class="sourceCode" id="cb67"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb67-1"><a href="primates.html#cb67-1" aria-hidden="true" tabindex="-1"></a>rootedml <span class="ot">&lt;-</span> <span class="fu">root</span>(treeml, </span>
<span id="cb67-2"><a href="primates.html#cb67-2" aria-hidden="true" tabindex="-1"></a>                 <span class="at">outgroup =</span> <span class="st">&quot;NC_013276.1 Mesocricetus auratus mitochondrion, complete genome&quot;</span>, </span>
<span id="cb67-3"><a href="primates.html#cb67-3" aria-hidden="true" tabindex="-1"></a>                 <span class="at">resolve.root =</span> <span class="cn">TRUE</span>) <span class="co">#rooting the tree</span></span>
<span id="cb67-4"><a href="primates.html#cb67-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb67-5"><a href="primates.html#cb67-5" aria-hidden="true" tabindex="-1"></a>rootedml<span class="sc">$</span>tip.label <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">&quot;.*</span><span class="sc">\\</span><span class="st">.[12]&quot;</span>, <span class="st">&quot;&quot;</span>, rootedml<span class="sc">$</span>tip.label) <span class="co">#clean up the labels</span></span>
<span id="cb67-6"><a href="primates.html#cb67-6" aria-hidden="true" tabindex="-1"></a>rootedml<span class="sc">$</span>tip.label <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">&quot;(mitochondrion, | isolate | voucher).*genome&quot;</span>, <span class="st">&quot;&quot;</span>, rootedml<span class="sc">$</span>tip.label) <span class="co">#clean up the labels</span></span>
<span id="cb67-7"><a href="primates.html#cb67-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb67-8"><a href="primates.html#cb67-8" aria-hidden="true" tabindex="-1"></a><span class="fu">ggtree</span>(rootedml, <span class="at">ladderize =</span> F)<span class="sc">+</span> <span class="co">#visualize the tree as a phylogenetic tree</span></span>
<span id="cb67-9"><a href="primates.html#cb67-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_tree</span>()<span class="sc">+</span></span>
<span id="cb67-10"><a href="primates.html#cb67-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_tiplab</span>(<span class="at">size =</span> <span class="fl">2.5</span>, <span class="at">as_ylab =</span> F)<span class="sc">+</span></span>
<span id="cb67-11"><a href="primates.html#cb67-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlim</span>(<span class="dv">0</span>, <span class="fl">0.35</span>)<span class="sc">+</span></span>
<span id="cb67-12"><a href="primates.html#cb67-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">&quot;Rooted maximum likelihood phylogenetic tree of</span><span class="sc">\n</span><span class="st">mitochondrial genomes of 15 primates and the golden hamster&quot;</span>)</span></code></pre></div>
<p><img src="portfolio_book_files/figure-html/Rooting%20the%20tree%20and%20visualizing%20it-1.png" width="672" /></p>
<p>The final tree is visualized as a phylogenetic tree instead of a cladogram, with different branch lengths describing the ancestry of the primates. The cladogram only showed how the different species are related <span class="citation">(<a href="#ref-pearsonWhenOutgroupsFail2013" role="doc-biblioref">Pearson et al. 2013</a>)</span>.</p>
<p>Normally, more thought would go into selecting the best possible outgroup <span class="citation">(<a href="#ref-rota-stabelliMultiCriterionApproach2008" role="doc-biblioref">Rota-Stabelli and Telford 2008</a>)</span>. Since my goal was to learn how to generate a phylogenetic tree in R (within a reasonable timeframe) I chose an animal that I like and know is more genetically different from the primates than the primates between each other.</p>

</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-anacondaMinicondaCondaDocumentation" class="csl-entry">
Anaconda. n.d. <span>“Miniconda — Conda Documentation.”</span> Accessed December 21, 2022. <a href="https://docs.conda.io/en/latest/miniconda.html">https://docs.conda.io/en/latest/miniconda.html</a>.
</div>
<div id="ref-bensonGenBank2013" class="csl-entry">
Benson, Dennis A., Mark Cavanaugh, Karen Clark, Ilene Karsch-Mizrachi, David J. Lipman, James Ostell, and Eric W. Sayers. 2013. <span>“GenBank.”</span> <em>Nucleic Acids Research</em> 41 (Database issue): D36–42. <a href="https://doi.org/10.1093/nar/gks1195">https://doi.org/10.1093/nar/gks1195</a>.
</div>
<div id="ref-cavanaughAkaikeInformationCriterion2019" class="csl-entry">
Cavanaugh, Joseph E., and Andrew A. Neath. 2019. <span>“The Akaike Information Criterion: Background, Derivation, Properties, Application, Interpretation, and Refinements.”</span> <em>WIREs Computational Statistics</em> 11 (3): e1460. <a href="https://doi.org/10.1002/wics.1460">https://doi.org/10.1002/wics.1460</a>.
</div>
<div id="ref-dharMaximumLikelihoodPhylogenetic2016" class="csl-entry">
Dhar, Amrit, and V.N. Minin. 2016. <span>“Maximum Likelihood Phylogenetic Inference.”</span> <em>Encyclopedia of Evolutionary Biology</em>, December. <a href="https://doi.org/10.1016/B978-0-12-800049-6.00207-9">https://doi.org/10.1016/B978-0-12-800049-6.00207-9</a>.
</div>
<div id="ref-gascuelBIONJImprovedVersion1997" class="csl-entry">
Gascuel, O. 1997. <span>“BIONJ: An Improved Version of the NJ Algorithm Based on a Simple Model of Sequence Data.”</span> <em>Molecular Biology and Evolution</em> 14 (7): 685–95. <a href="https://doi.org/10.1093/oxfordjournals.molbev.a025808">https://doi.org/10.1093/oxfordjournals.molbev.a025808</a>.
</div>
<div id="ref-jombartAdegenet31New2011" class="csl-entry">
Jombart, Thibaut, and Ismaïl Ahmed. 2011. <span>“Adegenet 1.3-1: New Tools for the Analysis of Genome-Wide SNP Data.”</span> <em>Bioinformatics</em> 27 (21): 3070–71. <a href="https://doi.org/10.1093/bioinformatics/btr521">https://doi.org/10.1093/bioinformatics/btr521</a>.
</div>
<div id="ref-kineneRootingTreesMethods2016" class="csl-entry">
Kinene, T., J. Wainaina, S. Maina, and L.M. Boykin. 2016. <span>“Rooting Trees, Methods For.”</span> <em>Encyclopedia of Evolutionary Biology</em>, 489–93. <a href="https://doi.org/10.1016/B978-0-12-800049-6.00215-8">https://doi.org/10.1016/B978-0-12-800049-6.00215-8</a>.
</div>
<div id="ref-mullerHere" class="csl-entry">
Müller, Kirill. n.d. <span>“Here.”</span> Accessed December 21, 2022. <a href="https://here.r-lib.org/articles/here.html">https://here.r-lib.org/articles/here.html</a>.
</div>
<div id="ref-ncbistaffNowAvailableUpdated2022" class="csl-entry">
NCBI Staff, NCBI. 2022. <span>“Now Available! Updated NCBI Datasets Command-Line Tools.”</span> NCBI Insights. October 12, 2022. <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2022/10/12/ncbi-datasets-command-line-tools/">https://ncbiinsights.ncbi.nlm.nih.gov/2022/10/12/ncbi-datasets-command-line-tools/</a>.
</div>
<div id="ref-paradisApeEnvironmentModern2019" class="csl-entry">
Paradis, Emmanuel, and Klaus Schliep. 2019. <span>“ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R.”</span> <em>Bioinformatics (Oxford, England)</em> 35 (3): 526–28. <a href="https://doi.org/10.1093/bioinformatics/bty633">https://doi.org/10.1093/bioinformatics/bty633</a>.
</div>
<div id="ref-pearsonWhenOutgroupsFail2013" class="csl-entry">
Pearson, Talima, Heidie M. Hornstra, Jason W. Sahl, Sarah Schaack, James M. Schupp, Stephen M. Beckstrom-Sternberg, Matthew W. O’Neill, et al. 2013. <span>“When outgroups fail; phylogenomics of rooting the emerging pathogen, Coxiella burnetii.”</span> <em>Systematic Biology</em> 62 (5): 752–62. <a href="https://doi.org/10.1093/sysbio/syt038">https://doi.org/10.1093/sysbio/syt038</a>.
</div>
<div id="ref-perelmanMolecularPhylogenyLiving2011" class="csl-entry">
Perelman, Polina, Warren E. Johnson, Christian Roos, Hector N. Seuánez, Julie E. Horvath, Miguel A. M. Moreira, Bailey Kessing, et al. 2011. <span>“A Molecular Phylogeny of Living Primates.”</span> <em>PLOS Genetics</em> 7 (3): e1001342. <a href="https://doi.org/10.1371/journal.pgen.1001342">https://doi.org/10.1371/journal.pgen.1001342</a>.
</div>
<div id="ref-pozziPrimatePhylogeneticRelationships2014" class="csl-entry">
Pozzi, Luca, Jason A. Hodgson, Andrew S. Burrell, Kirstin N. Sterner, Ryan L. Raaum, and Todd R. Disotell. 2014. <span>“Primate Phylogenetic Relationships and Divergence Dates Inferred from Complete Mitochondrial Genomes.”</span> <em>Molecular Phylogenetics and Evolution</em> 75 (June): 165–83. <a href="https://doi.org/10.1016/j.ympev.2014.02.023">https://doi.org/10.1016/j.ympev.2014.02.023</a>.
</div>
<div id="ref-rcoreteamLanguageEnvironmentStatistical2014" class="csl-entry">
R Core Team. 2014. <span>“R: A Language and Environment for Statistical Computing.”</span> <em>MSOR Connections</em>. <a href="https://www.semanticscholar.org/paper/R%3A-A-language-and-environment-for-statistical-Team/659408b243cec55de8d0a3bc51b81173007aa89b">https://www.semanticscholar.org/paper/R%3A-A-language-and-environment-for-statistical-Team/659408b243cec55de8d0a3bc51b81173007aa89b</a>.
</div>
<div id="ref-ramosInstallingBiocManager" class="csl-entry">
Ramos, Marcel, and Martin Morgan. n.d. <span>“Installing BiocManager.”</span> Accessed December 21, 2022. <a href="https://cran.r-project.org/web/packages/BiocManager/vignettes/BiocManager.html">https://cran.r-project.org/web/packages/BiocManager/vignettes/BiocManager.html</a>.
</div>
<div id="ref-rota-stabelliMultiCriterionApproach2008" class="csl-entry">
Rota-Stabelli, Omar, and Maximilian J. Telford. 2008. <span>“A Multi Criterion Approach for the Selection of Optimal Outgroups in Phylogeny: Recovering Some Support for Mandibulata over Myriochelata Using Mitogenomics.”</span> <em>Molecular Phylogenetics and Evolution</em> 48 (1): 103–11. <a href="https://doi.org/10.1016/j.ympev.2008.03.033">https://doi.org/10.1016/j.ympev.2008.03.033</a>.
</div>
<div id="ref-sieversFastScalableGeneration2011" class="csl-entry">
Sievers, Fabian, Andreas Wilm, David Dineen, Toby J. Gibson, Kevin Karplus, Weizhong Li, Rodrigo Lopez, et al. 2011. <span>“Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega.”</span> <em>Molecular Systems Biology</em> 7 (October): 539. <a href="https://doi.org/10.1038/msb.2011.75">https://doi.org/10.1038/msb.2011.75</a>.
</div>
<div id="ref-suchardBayesianPhylogeneticPhylodynamic2018" class="csl-entry">
Suchard, Marc A, Philippe Lemey, Guy Baele, Daniel L Ayres, Alexei J Drummond, and Andrew Rambaut. 2018. <span>“Bayesian Phylogenetic and Phylodynamic Data Integration Using BEAST 1.10.”</span> <em>Virus Evolution</em> 4 (1): vey016. <a href="https://doi.org/10.1093/ve/vey016">https://doi.org/10.1093/ve/vey016</a>.
</div>
<div id="ref-tamuraEstimationNumberNucleotide1993" class="csl-entry">
Tamura, K, and M Nei. 1993. <span>“Estimation of the Number of Nucleotide Substitutions in the Control Region of Mitochondrial DNA in Humans and Chimpanzees.”</span> <em>Molecular Biology and Evolution</em> 10 (3): 512–26. <a href="https://doi.org/10.1093/oxfordjournals.molbev.a040023">https://doi.org/10.1093/oxfordjournals.molbev.a040023</a>.
</div>
<div id="ref-vaidyaSequenceMatrixConcatenationSoftware2011" class="csl-entry">
Vaidya, Gaurav, David J. Lohman, and Rudolf Meier. 2011. <span>“SequenceMatrix: Concatenation Software for the Fast Assembly of Multi-Gene Datasets with Character Set and Codon Information.”</span> <em>Cladistics</em> 27 (2): 171–80. <a href="https://doi.org/10.1111/j.1096-0031.2010.00329.x">https://doi.org/10.1111/j.1096-0031.2010.00329.x</a>.
</div>
<div id="ref-wickhamWelcomeTidyverse2019" class="csl-entry">
Wickham, Hadley, Mara Averick, Jennifer Bryan, Winston Chang, Lucy D’Agostino McGowan, Romain François, Garrett Grolemund, et al. 2019. <span>“Welcome to the Tidyverse.”</span> <em>Journal of Open Source Software</em> 4 (43): 1686. <a href="https://doi.org/10.21105/joss.01686">https://doi.org/10.21105/joss.01686</a>.
</div>
<div id="ref-winterRentrezPackageNCBI2017" class="csl-entry">
Winter, David J. 2017. <span>“Rentrez: An R Package for the NCBI eUtils API”</span> 9.
</div>
<div id="ref-yoshidaEfficienciesNJpMaximum2016" class="csl-entry">
Yoshida, Ruriko, and Masatoshi Nei. 2016. <span>“Efficiencies of the NJp, Maximum Likelihood, and Bayesian Methods of Phylogenetic Construction for Compositional and Noncompositional Genes.”</span> <em>Molecular Biology and Evolution</em> 33 (6): 1618–24. <a href="https://doi.org/10.1093/molbev/msw042">https://doi.org/10.1093/molbev/msw042</a>.
</div>
<div id="ref-yuGgtreePackageVisualization2017" class="csl-entry">
Yu, Guangchuang, David K. Smith, Huachen Zhu, Yi Guan, and Tommy Tsan-Yuk Lam. 2017. <span>“Ggtree: An r Package for Visualization and Annotation of Phylogenetic Trees with Their Covariates and Other Associated Data.”</span> <em>Methods in Ecology and Evolution</em> 8 (1): 28–36. <a href="https://doi.org/10.1111/2041-210X.12628">https://doi.org/10.1111/2041-210X.12628</a>.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="rpackage.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="references.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
}
});
});
</script>

</body>

</html>
