<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>References | portfolio_book.knit</title>
  <meta name="description" content="" />
  <meta name="generator" content="bookdown 0.31 and GitBook 2.6.7" />

  <meta property="og:title" content="References | portfolio_book.knit" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="References | portfolio_book.knit" />
  
  
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="primates.html"/>

<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<script src="libs/htmlwidgets-1.6.1/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.11.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.11.1/plotly-latest.min.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="booktheme.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<div class="titleandname">Portfolio Nine Luijendijk</div>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Curriculum Vitae</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#contact"><i class="fa fa-check"></i>Contact</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#education"><i class="fa fa-check"></i>Education</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#experience"><i class="fa fa-check"></i>Experience</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#skills"><i class="fa fa-check"></i>Skills</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#languages"><i class="fa fa-check"></i>Languages</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="guerillaframework.html"><a href="guerillaframework.html"><i class="fa fa-check"></i><b>1</b> Guerilla Analytics framework</a></li>
<li class="chapter" data-level="2" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html"><i class="fa fa-check"></i><b>2</b> Reproducible Research</a>
<ul>
<li class="chapter" data-level="2.1" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#scoring-the-reproducibility-of-a-research-article"><i class="fa fa-check"></i><b>2.1</b> Scoring the reproducibility of a research article</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#reproducible-research-and-criteria"><i class="fa fa-check"></i><b>2.1.1</b> Reproducible research and criteria</a></li>
<li class="chapter" data-level="2.1.2" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#the-research-article"><i class="fa fa-check"></i><b>2.1.2</b> The research article</a></li>
<li class="chapter" data-level="2.1.3" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#reproducibility-of-the-research"><i class="fa fa-check"></i><b>2.1.3</b> Reproducibility of the research</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#scoring-the-reproducibility-of-the-code-of-the-research"><i class="fa fa-check"></i><b>2.2</b> Scoring the reproducibility of the code of the research</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#criteria-for-the-code"><i class="fa fa-check"></i><b>2.2.1</b> Criteria for the code</a></li>
<li class="chapter" data-level="2.2.2" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#the-code"><i class="fa fa-check"></i><b>2.2.2</b> The code</a></li>
<li class="chapter" data-level="" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#install-andor-load-packages"><i class="fa fa-check"></i>Install and/or load packages</a></li>
<li class="chapter" data-level="" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#data"><i class="fa fa-check"></i>Data</a></li>
<li class="chapter" data-level="" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#multiple-item-psychological-measures"><i class="fa fa-check"></i>Multiple-item Psychological Measures</a></li>
<li class="chapter" data-level="" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#install-andor-load-packages-1"><i class="fa fa-check"></i>Install and/or load packages</a></li>
<li class="chapter" data-level="" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#data-1"><i class="fa fa-check"></i>Data</a></li>
<li class="chapter" data-level="" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#multiple-item-psychological-measures-1"><i class="fa fa-check"></i>Multiple-item Psychological Measures</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#final-rating"><i class="fa fa-check"></i><b>2.3</b> Final rating</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="datanalysis.html"><a href="datanalysis.html"><i class="fa fa-check"></i><b>3</b> Analyzing and visualizing data</a>
<ul>
<li class="chapter" data-level="3.1" data-path="datanalysis.html"><a href="datanalysis.html#preparing-the-data"><i class="fa fa-check"></i><b>3.1</b> Preparing the data</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="datanalysis.html"><a href="datanalysis.html#inspecting-the-data-in-r"><i class="fa fa-check"></i><b>3.1.1</b> Inspecting the data in R</a></li>
<li class="chapter" data-level="3.1.2" data-path="datanalysis.html"><a href="datanalysis.html#fixing-the-characterdouble-issue"><i class="fa fa-check"></i><b>3.1.2</b> Fixing the character/double issue</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="datanalysis.html"><a href="datanalysis.html#analyzing-and-visualizing-the-data"><i class="fa fa-check"></i><b>3.2</b> Analyzing and visualizing the data</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="datanalysis.html"><a href="datanalysis.html#plan-for-ic50ed50-visualization"><i class="fa fa-check"></i><b>3.2.1</b> Plan for IC50/ED50 visualization</a></li>
<li class="chapter" data-level="3.2.2" data-path="datanalysis.html"><a href="datanalysis.html#normalizing-the-data"><i class="fa fa-check"></i><b>3.2.2</b> Normalizing the data</a></li>
<li class="chapter" data-level="3.2.3" data-path="datanalysis.html"><a href="datanalysis.html#creating-a-legend"><i class="fa fa-check"></i><b>3.2.3</b> Creating a legend</a></li>
<li class="chapter" data-level="3.2.4" data-path="datanalysis.html"><a href="datanalysis.html#generating-the-normalized-plot"><i class="fa fa-check"></i><b>3.2.4</b> Generating the normalized plot</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="drugscreening.html"><a href="drugscreening.html"><i class="fa fa-check"></i><b>4</b> Drug Screening</a></li>
<li class="chapter" data-level="5" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html"><i class="fa fa-check"></i><b>5</b> Working with relational databases</a>
<ul>
<li class="chapter" data-level="5.1" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#preparing-the-data-1"><i class="fa fa-check"></i><b>5.1</b> Preparing the data</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#introduction"><i class="fa fa-check"></i><b>5.1.1</b> Introduction</a></li>
<li class="chapter" data-level="5.1.2" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#importing-the-data-and-making-it-tidy"><i class="fa fa-check"></i><b>5.1.2</b> Importing the data and making it tidy</a></li>
<li class="chapter" data-level="5.1.3" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#connecting-to-the-database"><i class="fa fa-check"></i><b>5.1.3</b> Connecting to the database</a></li>
<li class="chapter" data-level="5.1.4" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#exporting-the-tidy-dataframes-to-the-database"><i class="fa fa-check"></i><b>5.1.4</b> Exporting the tidy dataframes to the database</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#inspecting-the-data"><i class="fa fa-check"></i><b>5.2</b> Inspecting the data</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#inspecting-the-data-using-sql"><i class="fa fa-check"></i><b>5.2.1</b> Inspecting the data using SQL</a></li>
<li class="chapter" data-level="5.2.2" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#inspecting-the-data-using-r"><i class="fa fa-check"></i><b>5.2.2</b> Inspecting the data using R</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#combining-the-dataframes"><i class="fa fa-check"></i><b>5.3</b> Combining the dataframes</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#preparing-the-tidy-dataframes"><i class="fa fa-check"></i><b>5.3.1</b> Preparing the tidy dataframes</a></li>
<li class="chapter" data-level="5.3.2" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#joining-the-dataframes-together"><i class="fa fa-check"></i><b>5.3.2</b> Joining the dataframes together</a></li>
<li class="chapter" data-level="5.3.3" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#loading-the-combined-tables-into-r"><i class="fa fa-check"></i><b>5.3.3</b> Loading the combined tables into R</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#visualizing-the-data"><i class="fa fa-check"></i><b>5.4</b> Visualizing the data</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#normalizing-the-data-1"><i class="fa fa-check"></i><b>5.4.1</b> Normalizing the data</a></li>
<li class="chapter" data-level="5.4.2" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#calculating-descriptive-statistics"><i class="fa fa-check"></i><b>5.4.2</b> Calculating descriptive statistics</a></li>
<li class="chapter" data-level="5.4.3" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#visualizing-correlation"><i class="fa fa-check"></i><b>5.4.3</b> Visualizing correlation</a></li>
<li class="chapter" data-level="5.4.4" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#visualizing-the-dengue-data"><i class="fa fa-check"></i><b>5.4.4</b> Visualizing the dengue data</a></li>
<li class="chapter" data-level="5.4.5" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#visualizing-the-flu-data"><i class="fa fa-check"></i><b>5.4.5</b> Visualizing the flu data</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="parameterized.html"><a href="parameterized.html"><i class="fa fa-check"></i><b>6</b> Parameterized report on COVID-19</a>
<ul>
<li class="chapter" data-level="6.1" data-path="parameterized.html"><a href="parameterized.html#parameterized-reports"><i class="fa fa-check"></i><b>6.1</b> Parameterized reports</a></li>
<li class="chapter" data-level="6.2" data-path="parameterized.html"><a href="parameterized.html#the-parameterized-covid-19-report"><i class="fa fa-check"></i><b>6.2</b> The parameterized COVID-19 report</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="parameterized.html"><a href="parameterized.html#the-parameters"><i class="fa fa-check"></i><b>6.2.1</b> The parameters</a></li>
<li class="chapter" data-level="6.2.2" data-path="parameterized.html"><a href="parameterized.html#rendering-the-report"><i class="fa fa-check"></i><b>6.2.2</b> Rendering the report</a></li>
<li class="chapter" data-level="6.2.3" data-path="parameterized.html"><a href="parameterized.html#using-different-parameters"><i class="fa fa-check"></i><b>6.2.3</b> Using different parameters</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="vignette.html"><a href="vignette.html"><i class="fa fa-check"></i><b>7</b> R package vignette</a>
<ul>
<li class="chapter" data-level="7.1" data-path="vignette.html"><a href="vignette.html#creating-the-package"><i class="fa fa-check"></i><b>7.1</b> Creating the package</a></li>
<li class="chapter" data-level="7.2" data-path="vignette.html"><a href="vignette.html#installing-and-loading-the-package"><i class="fa fa-check"></i><b>7.2</b> Installing and loading the package</a></li>
<li class="chapter" data-level="7.3" data-path="vignette.html"><a href="vignette.html#package-contents"><i class="fa fa-check"></i><b>7.3</b> Package contents</a>
<ul>
<li class="chapter" data-level="7.3.1" data-path="vignette.html"><a href="vignette.html#dataset-measlesdata"><i class="fa fa-check"></i><b>7.3.1</b> Dataset measlesdata</a></li>
<li class="chapter" data-level="7.3.2" data-path="vignette.html"><a href="vignette.html#function-datasummarizer"><i class="fa fa-check"></i><b>7.3.2</b> Function dataSummarizer()</a></li>
<li class="chapter" data-level="7.3.3" data-path="vignette.html"><a href="vignette.html#function-illnessplot"><i class="fa fa-check"></i><b>7.3.3</b> Function illnessPlot()</a></li>
<li class="chapter" data-level="7.3.4" data-path="vignette.html"><a href="vignette.html#function-fastaimporter.r"><i class="fa fa-check"></i><b>7.3.4</b> Function fastaImporter.R</a></li>
<li class="chapter" data-level="7.3.5" data-path="vignette.html"><a href="vignette.html#function-gotermanalysis"><i class="fa fa-check"></i><b>7.3.5</b> Function gotermAnalysis()</a></li>
<li class="chapter" data-level="7.3.6" data-path="vignette.html"><a href="vignette.html#function-gotermplot"><i class="fa fa-check"></i><b>7.3.6</b> Function gotermPlot()</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="vignette.html"><a href="vignette.html#additional-files"><i class="fa fa-check"></i><b>7.4</b> Additional files</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="primates.html"><a href="primates.html"><i class="fa fa-check"></i><b>8</b> Generating a phylogenetic tree in R</a>
<ul>
<li class="chapter" data-level="8.1" data-path="primates.html"><a href="primates.html#the-plan"><i class="fa fa-check"></i><b>8.1</b> The plan</a></li>
<li class="chapter" data-level="8.2" data-path="primates.html"><a href="primates.html#preparatory-research"><i class="fa fa-check"></i><b>8.2</b> Preparatory research</a></li>
<li class="chapter" data-level="8.3" data-path="primates.html"><a href="primates.html#getting-started"><i class="fa fa-check"></i><b>8.3</b> Getting started</a></li>
<li class="chapter" data-level="8.4" data-path="primates.html"><a href="primates.html#generating-the-first-tree"><i class="fa fa-check"></i><b>8.4</b> Generating the first tree</a>
<ul>
<li class="chapter" data-level="8.4.1" data-path="primates.html"><a href="primates.html#obtaining-fasta-sequences"><i class="fa fa-check"></i><b>8.4.1</b> Obtaining FASTA sequences</a></li>
<li class="chapter" data-level="8.4.2" data-path="primates.html"><a href="primates.html#performing-multiple-sequence-alignment"><i class="fa fa-check"></i><b>8.4.2</b> Performing multiple sequence alignment</a></li>
<li class="chapter" data-level="8.4.3" data-path="primates.html"><a href="primates.html#importing-the-aligned-dna"><i class="fa fa-check"></i><b>8.4.3</b> Importing the aligned dna</a></li>
<li class="chapter" data-level="8.4.4" data-path="primates.html"><a href="primates.html#calculating-the-genetic-distances-between-the-sequences-and-drawing-a-tree"><i class="fa fa-check"></i><b>8.4.4</b> Calculating the genetic distances between the sequences and drawing a tree</a></li>
</ul></li>
<li class="chapter" data-level="8.5" data-path="primates.html"><a href="primates.html#different-types-of-phylogenetic-models"><i class="fa fa-check"></i><b>8.5</b> Different types of phylogenetic models</a>
<ul>
<li class="chapter" data-level="8.5.1" data-path="primates.html"><a href="primates.html#finding-the-best-model"><i class="fa fa-check"></i><b>8.5.1</b> Finding the best model</a></li>
<li class="chapter" data-level="8.5.2" data-path="primates.html"><a href="primates.html#generating-a-maximum-likelihood-tree"><i class="fa fa-check"></i><b>8.5.2</b> Generating a maximum likelihood tree</a></li>
<li class="chapter" data-level="8.5.3" data-path="primates.html"><a href="primates.html#visualizing-the-maximum-likelihood-cladogram"><i class="fa fa-check"></i><b>8.5.3</b> Visualizing the maximum likelihood cladogram</a></li>
<li class="chapter" data-level="8.5.4" data-path="primates.html"><a href="primates.html#rooting-the-tree"><i class="fa fa-check"></i><b>8.5.4</b> Rooting the tree</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<hr />

<div id="refs" class="references csl-bib-body hanging-indent">
<div class="csl-entry">
Anaconda. n.d. <span>“Miniconda — Conda Documentation.”</span> Accessed December 21, 2022. <a href="https://docs.conda.io/en/latest/miniconda.html">https://docs.conda.io/en/latest/miniconda.html</a>.
</div>
<div class="csl-entry">
Benson, Dennis A., Mark Cavanaugh, Karen Clark, Ilene Karsch-Mizrachi, David J. Lipman, James Ostell, and Eric W. Sayers. 2013. <span>“GenBank.”</span> <em>Nucleic Acids Research</em> 41 (Database issue): D36–42. <a href="https://doi.org/10.1093/nar/gks1195">https://doi.org/10.1093/nar/gks1195</a>.
</div>
<div class="csl-entry">
Cavanaugh, Joseph E., and Andrew A. Neath. 2019. <span>“The Akaike Information Criterion: Background, Derivation, Properties, Application, Interpretation, and Refinements.”</span> <em>WIREs Computational Statistics</em> 11 (3): e1460. <a href="https://doi.org/10.1002/wics.1460">https://doi.org/10.1002/wics.1460</a>.
</div>
<div class="csl-entry">
Coen A. Bernaards, and Robert I. Jennrich. 2005. <span>“Gradient Projection Algorithms and Software for Arbitrary Rotation Criteria in Factor Analysis.”</span> <em>Educational and Psychological Measurement</em> 65: 676--696.
</div>
<div class="csl-entry">
Cokol, Murat. 2012. <span>“Drugs and Their Interactions.”</span> <em>Current Drug Discovery Technologies</em> 10 (December). <a href="https://doi.org/10.2174/1570163811310020003">https://doi.org/10.2174/1570163811310020003</a>.
</div>
<div class="csl-entry">
Dhar, Amrit, and V.N. Minin. 2016. <span>“Maximum Likelihood Phylogenetic Inference.”</span> <em>Encyclopedia of Evolutionary Biology</em>, December. <a href="https://doi.org/10.1016/B978-0-12-800049-6.00207-9">https://doi.org/10.1016/B978-0-12-800049-6.00207-9</a>.
</div>
<div class="csl-entry">
Gascuel, O. 1997. <span>“BIONJ: An Improved Version of the NJ Algorithm Based on a Simple Model of Sequence Data.”</span> <em>Molecular Biology and Evolution</em> 14 (7): 685–95. <a href="https://doi.org/10.1093/oxfordjournals.molbev.a025808">https://doi.org/10.1093/oxfordjournals.molbev.a025808</a>.
</div>
<div class="csl-entry">
Gubler, Duane J. 2002. <span>“The Global Emergence/Resurgence of Arboviral Diseases As Public Health Problems.”</span> <em>Archives of Medical Research</em> 33 (4): 330–42. <a href="https://doi.org/10.1016/S0188-4409(02)00378-8">https://doi.org/10.1016/S0188-4409(02)00378-8</a>.
</div>
<div class="csl-entry">
<span>“High-Throughput Screening (HTS).”</span> n.d. Research. Accessed January 1, 2023. <a href="https://research.prinsesmaximacentrum.nl/en/core-facilities/high-throughput-screening">https://research.prinsesmaximacentrum.nl/en/core-facilities/high-throughput-screening</a>.
</div>
<div class="csl-entry">
Irizarry, Rafael A., and Amy Gill. 2021. <span>“Dslabs: Data Science Labs.”</span> <a href="https://CRAN.R-project.org/package=dslabs">https://CRAN.R-project.org/package=dslabs</a>.
</div>
<div class="csl-entry">
Jaaks, Patricia, Elizabeth A. Coker, Daniel J. Vis, Olivia Edwards, Emma F. Carpenter, Simonetta M. Leto, Lisa Dwane, et al. 2022. <span>“Effective Drug Combinations in Breast, Colon and Pancreatic Cancer Cells.”</span> <em>Nature</em> 603 (7899): 166–73. <a href="https://doi.org/10.1038/s41586-022-04437-2">https://doi.org/10.1038/s41586-022-04437-2</a>.
</div>
<div class="csl-entry">
Jombart, Thibaut, and Ismaïl Ahmed. 2011. <span>“Adegenet 1.3-1: New Tools for the Analysis of Genome-Wide SNP Data.”</span> <em>Bioinformatics</em> 27 (21): 3070–71. <a href="https://doi.org/10.1093/bioinformatics/btr521">https://doi.org/10.1093/bioinformatics/btr521</a>.
</div>
<div class="csl-entry">
Kinene, T., J. Wainaina, S. Maina, and L.M. Boykin. 2016. <span>“Rooting Trees, Methods For.”</span> <em>Encyclopedia of Evolutionary Biology</em>, 489–93. <a href="https://doi.org/10.1016/B978-0-12-800049-6.00215-8">https://doi.org/10.1016/B978-0-12-800049-6.00215-8</a>.
</div>
<div class="csl-entry">
Lewandowska, Anna, Barbara Zych, Katalin Papp, Dana Zrubcová, Helena Kadučáková, Mária Šupínová, Serap Ejder Apay, and Małgorzata Nagórska. 2021. <span>“Problems, Stressors and Needs of Children and Adolescents with Cancer.”</span> <em>Children</em> 8 (12): 1173. <a href="https://doi.org/10.3390/children8121173">https://doi.org/10.3390/children8121173</a>.
</div>
<div class="csl-entry">
Michalak, Nicholas M., Oliver Sng, Iris M. Wang, and Joshua Ackerman. 2020. <span>“Sounds of Sickness: Can People Identify Infectious Disease Using Sounds of Coughs and Sneezes? | Proceedings of the Royal Society B: Biological Sciences.”</span> June 10, 2020. <a href="https://doi.org/10.1098/rspb.2020.0944">https://doi.org/10.1098/rspb.2020.0944</a>.
</div>
<div class="csl-entry">
Müller, Kirill. n.d. <span>“Here.”</span> Accessed December 21, 2022. <a href="https://here.r-lib.org/articles/here.html">https://here.r-lib.org/articles/here.html</a>.
</div>
<div class="csl-entry">
NCBI Staff, NCBI. 2022. <span>“Now Available! Updated NCBI Datasets Command-Line Tools.”</span> NCBI Insights. October 12, 2022. <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2022/10/12/ncbi-datasets-command-line-tools/">https://ncbiinsights.ncbi.nlm.nih.gov/2022/10/12/ncbi-datasets-command-line-tools/</a>.
</div>
<div class="csl-entry">
Paradis, Emmanuel, and Klaus Schliep. 2019. <span>“ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R.”</span> <em>Bioinformatics (Oxford, England)</em> 35 (3): 526–28. <a href="https://doi.org/10.1093/bioinformatics/bty633">https://doi.org/10.1093/bioinformatics/bty633</a>.
</div>
<div class="csl-entry">
Pearson, Talima, Heidie M. Hornstra, Jason W. Sahl, Sarah Schaack, James M. Schupp, Stephen M. Beckstrom-Sternberg, Matthew W. O’Neill, et al. 2013. <span>“When outgroups fail; phylogenomics of rooting the emerging pathogen, Coxiella burnetii.”</span> <em>Systematic Biology</em> 62 (5): 752–62. <a href="https://doi.org/10.1093/sysbio/syt038">https://doi.org/10.1093/sysbio/syt038</a>.
</div>
<div class="csl-entry">
Perelman, Polina, Warren E. Johnson, Christian Roos, Hector N. Seuánez, Julie E. Horvath, Miguel A. M. Moreira, Bailey Kessing, et al. 2011. <span>“A Molecular Phylogeny of Living Primates.”</span> <em>PLOS Genetics</em> 7 (3): e1001342. <a href="https://doi.org/10.1371/journal.pgen.1001342">https://doi.org/10.1371/journal.pgen.1001342</a>.
</div>
<div class="csl-entry">
Pozzi, Luca, Jason A. Hodgson, Andrew S. Burrell, Kirstin N. Sterner, Ryan L. Raaum, and Todd R. Disotell. 2014. <span>“Primate Phylogenetic Relationships and Divergence Dates Inferred from Complete Mitochondrial Genomes.”</span> <em>Molecular Phylogenetics and Evolution</em> 75 (June): 165–83. <a href="https://doi.org/10.1016/j.ympev.2014.02.023">https://doi.org/10.1016/j.ympev.2014.02.023</a>.
</div>
<div class="csl-entry">
Prinses Máxima Centrum. n.d. <span>“Ons verhaal.”</span> Prinses Máxima Centrum. Accessed January 1, 2023. <a href="https://www.prinsesmaximacentrum.nl/nl/over-ons/ons-verhaal">https://www.prinsesmaximacentrum.nl/nl/over-ons/ons-verhaal</a>.
</div>
<div class="csl-entry">
R Core Team. 2014. <span>“R: A Language and Environment for Statistical Computing.”</span> <em>MSOR Connections</em>. <a href="https://www.semanticscholar.org/paper/R%3A-A-language-and-environment-for-statistical-Team/659408b243cec55de8d0a3bc51b81173007aa89b">https://www.semanticscholar.org/paper/R%3A-A-language-and-environment-for-statistical-Team/659408b243cec55de8d0a3bc51b81173007aa89b</a>.
</div>
<div class="csl-entry">
Ramos, Marcel, and Martin Morgan. n.d. <span>“Installing BiocManager.”</span> Accessed December 21, 2022. <a href="https://cran.r-project.org/web/packages/BiocManager/vignettes/BiocManager.html">https://cran.r-project.org/web/packages/BiocManager/vignettes/BiocManager.html</a>.
</div>
<div class="csl-entry">
Ridge, Enda. 2014. <span>“Guerrilla Analytics - 1st Edition.”</span> 2014. <a href="https://www.elsevier.com/books/T/A/9780128002186">https://www.elsevier.com/books/T/A/9780128002186</a>.
</div>
<div class="csl-entry">
Ritz, Christian, Florent Baty, Jens C. Streibig, and Daniel Gerhard. 2015. <span>“Dose-Response Analysis Using R.”</span> <em>PLOS ONE</em> 10 (12): e0146021. <a href="https://doi.org/10.1371/journal.pone.0146021">https://doi.org/10.1371/journal.pone.0146021</a>.
</div>
<div class="csl-entry">
Rota-Stabelli, Omar, and Maximilian J. Telford. 2008. <span>“A Multi Criterion Approach for the Selection of Optimal Outgroups in Phylogeny: Recovering Some Support for Mandibulata over Myriochelata Using Mitogenomics.”</span> <em>Molecular Phylogenetics and Evolution</em> 48 (1): 103–11. <a href="https://doi.org/10.1016/j.ympev.2008.03.033">https://doi.org/10.1016/j.ympev.2008.03.033</a>.
</div>
<div class="csl-entry">
Rounsaville, Bruce J., Nancy M. Petry, and Kathleen M. Carroll. 2003. <span>“Single Versus Multiple Drug Focus in Substance Abuse Clinical Trials Research.”</span> <em>Drug and Alcohol Dependence</em> 70 (2): 117–25. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3662471/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3662471/</a>.
</div>
<div class="csl-entry">
Schüz, Joachim, and Eve Roman. 2021. <span>“Childhood Cancer: A Global Perspective.”</span> <em>Cancer Epidemiology</em>, Childhood Cancer: A Global Perspective, 71 (April): 101878. <a href="https://doi.org/10.1016/j.canep.2020.101878">https://doi.org/10.1016/j.canep.2020.101878</a>.
</div>
<div class="csl-entry">
Sievers, Fabian, Andreas Wilm, David Dineen, Toby J. Gibson, Kevin Karplus, Weizhong Li, Rodrigo Lopez, et al. 2011. <span>“Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega.”</span> <em>Molecular Systems Biology</em> 7 (October): 539. <a href="https://doi.org/10.1038/msb.2011.75">https://doi.org/10.1038/msb.2011.75</a>.
</div>
<div class="csl-entry">
SteliarovaFoucher, Eva, Charles Stiller, Peter Kaatsch, Franco Berrino, Jan-Willem Coebergh, Brigitte Lacour, and Max Parkin. 2004. <span>“Geographical patterns and time trends of cancer incidence and survival among children and adolescents in Europe since the 1970s (the ACCISproject): an epidemiological study.”</span> <em>Lancet (London, England)</em> 364 (9451): 2097–2105. <a href="https://doi.org/10.1016/s0140-6736(04)17550-8">https://doi.org/10.1016/s0140-6736(04)17550-8</a>.
</div>
<div class="csl-entry">
Suchard, Marc A, Philippe Lemey, Guy Baele, Daniel L Ayres, Alexei J Drummond, and Andrew Rambaut. 2018. <span>“Bayesian Phylogenetic and Phylodynamic Data Integration Using BEAST 1.10.”</span> <em>Virus Evolution</em> 4 (1): vey016. <a href="https://doi.org/10.1093/ve/vey016">https://doi.org/10.1093/ve/vey016</a>.
</div>
<div class="csl-entry">
Sumner, Josh, Leah Haynes, Sarah Nathan, Cynthia Hudson-Vitale, and Leslie D. McIntosh. 2020. <span>“Reproducibility and Reporting Practices in COVID-19 Preprint Manuscripts.”</span> medRxiv. <a href="https://doi.org/10.1101/2020.03.24.20042796">https://doi.org/10.1101/2020.03.24.20042796</a>.
</div>
<div class="csl-entry">
Tamura, K, and M Nei. 1993. <span>“Estimation of the Number of Nucleotide Substitutions in the Control Region of Mitochondrial DNA in Humans and Chimpanzees.”</span> <em>Molecular Biology and Evolution</em> 10 (3): 512–26. <a href="https://doi.org/10.1093/oxfordjournals.molbev.a040023">https://doi.org/10.1093/oxfordjournals.molbev.a040023</a>.
</div>
<div class="csl-entry">
Taylor, Richard. 1990. <span>“Interpretation of the Correlation Coefficient: A Basic Review.”</span> <em>Journal of Diagnostic Medical Sonography</em> 6 (1): 35–39. <a href="https://doi.org/10.1177/875647939000600106">https://doi.org/10.1177/875647939000600106</a>.
</div>
<div class="csl-entry">
Vaidya, Gaurav, David J. Lohman, and Rudolf Meier. 2011. <span>“SequenceMatrix: Concatenation Software for the Fast Assembly of Multi-Gene Datasets with Character Set and Codon Information.”</span> <em>Cladistics</em> 27 (2): 171–80. <a href="https://doi.org/10.1111/j.1096-0031.2010.00329.x">https://doi.org/10.1111/j.1096-0031.2010.00329.x</a>.
</div>
<div class="csl-entry">
Wickham, Hadley. 2016. <span>“Ggplot2: Elegant Graphics for Data Analysis.”</span> 2016. <a href="https://ggplot2.tidyverse.org">https://ggplot2.tidyverse.org</a>.
</div>
<div class="csl-entry">
Wickham, Hadley, Mara Averick, Jennifer Bryan, Winston Chang, Lucy D’Agostino McGowan, Romain François, Garrett Grolemund, et al. 2019. <span>“Welcome to the Tidyverse.”</span> <em>Journal of Open Source Software</em> 4 (43): 1686. <a href="https://doi.org/10.21105/joss.01686">https://doi.org/10.21105/joss.01686</a>.
</div>
<div class="csl-entry">
Wickham, Hadley, Jennifer Bryan, and Malcom Barrett. 2022. <span>“Usethis: Automate Package and Project Setup.”</span> <a href="https://usethis.r-lib.org, https://github.com/r-lib/usethis">https://usethis.r-lib.org, https://github.com/r-lib/usethis</a>.
</div>
<div class="csl-entry">
Wickham, Hadley, Jim Hester, Winston Chang, and Jennifer Bryan. 2022. <span>“Devtools: Tools to Make Developing R Packages Easier.”</span> <a href="https://devtools.r-lib.org/, https://github.com/r-lib/devtools">https://devtools.r-lib.org/, https://github.com/r-lib/devtools</a>.
</div>
<div class="csl-entry">
Wickham, H, J Ooms, and K Müller. 2022. <span>“RPostgres: Rcpp Interface to PostgreSQL.”</span> <a href="https://rpostgres.r-dbi.org">https://rpostgres.r-dbi.org</a>.
</div>
<div class="csl-entry">
Winter, David J. 2017. <span>“Rentrez: An R Package for the NCBI eUtils API”</span> 9.
</div>
<div class="csl-entry">
Yoshida, Ruriko, and Masatoshi Nei. 2016. <span>“Efficiencies of the NJp, Maximum Likelihood, and Bayesian Methods of Phylogenetic Construction for Compositional and Noncompositional Genes.”</span> <em>Molecular Biology and Evolution</em> 33 (6): 1618–24. <a href="https://doi.org/10.1093/molbev/msw042">https://doi.org/10.1093/molbev/msw042</a>.
</div>
<div class="csl-entry">
Yu, Guangchuang, David K. Smith, Huachen Zhu, Yi Guan, and Tommy Tsan-Yuk Lam. 2017. <span>“Ggtree: An r Package for Visualization and Annotation of Phylogenetic Trees with Their Covariates and Other Associated Data.”</span> <em>Methods in Ecology and Evolution</em> 8 (1): 28–36. <a href="https://doi.org/10.1111/2041-210X.12628">https://doi.org/10.1111/2041-210X.12628</a>.
</div>
<div class="csl-entry">
Zheng, Shuyu, Wenyu Wang, Jehad Aldahdooh, Alina Malyutina, Tolou Shadbahr, Ziaurrehman Tanoli, Alberto Pessia, and Jing Tang. 2022. <span>“SynergyFinder Plus: Toward Better Interpretation and Annotation of Drug Combination Screening Datasets.”</span> <em>Genomics, Proteomics &amp; Bioinformatics</em>, January. <a href="https://doi.org/10.1016/j.gpb.2022.01.004">https://doi.org/10.1016/j.gpb.2022.01.004</a>.
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="primates.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["portfolio_book.pdf"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
}
});
});
</script>

</body>

</html>
