<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>7 R package vignette | portfolio_book.knit</title>
  <meta name="description" content="" />
  <meta name="generator" content="bookdown 0.31 and GitBook 2.6.7" />

  <meta property="og:title" content="7 R package vignette | portfolio_book.knit" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="7 R package vignette | portfolio_book.knit" />
  
  
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="parameterized.html"/>
<link rel="next" href="primates.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<script src="libs/htmlwidgets-1.6.0/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.11.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.11.1/plotly-latest.min.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="booktheme.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<div class="titleandname">Portfolio Nine Luijendijk</div>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Curriculum Vitae</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#contact"><i class="fa fa-check"></i>Contact</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#education"><i class="fa fa-check"></i>Education</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#experience"><i class="fa fa-check"></i>Experience</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#skills"><i class="fa fa-check"></i>Skills</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#languages"><i class="fa fa-check"></i>Languages</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="guerillaframework.html"><a href="guerillaframework.html"><i class="fa fa-check"></i><b>1</b> Guerilla Analytics framework</a></li>
<li class="chapter" data-level="2" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html"><i class="fa fa-check"></i><b>2</b> Reproducible Research</a>
<ul>
<li class="chapter" data-level="2.1" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#scoring-the-reproducibility-of-a-research-article"><i class="fa fa-check"></i><b>2.1</b> Scoring the reproducibility of a research article</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#reproducible-research-and-criteria"><i class="fa fa-check"></i><b>2.1.1</b> Reproducible research and criteria</a></li>
<li class="chapter" data-level="2.1.2" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#the-research-article"><i class="fa fa-check"></i><b>2.1.2</b> The research article</a></li>
<li class="chapter" data-level="2.1.3" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#reproducibility-of-the-research"><i class="fa fa-check"></i><b>2.1.3</b> Reproducibility of the research</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#scoring-the-reproducibility-of-the-code-of-the-research"><i class="fa fa-check"></i><b>2.2</b> Scoring the reproducibility of the code of the research</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#criteria-for-the-code"><i class="fa fa-check"></i><b>2.2.1</b> Criteria for the code</a></li>
<li class="chapter" data-level="2.2.2" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#the-code"><i class="fa fa-check"></i><b>2.2.2</b> The code</a></li>
<li class="chapter" data-level="" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#install-andor-load-packages"><i class="fa fa-check"></i>Install and/or load packages</a></li>
<li class="chapter" data-level="" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#data"><i class="fa fa-check"></i>Data</a></li>
<li class="chapter" data-level="" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#multiple-item-psychological-measures"><i class="fa fa-check"></i>Multiple-item Psychological Measures</a></li>
<li class="chapter" data-level="" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#install-andor-load-packages-1"><i class="fa fa-check"></i>Install and/or load packages</a></li>
<li class="chapter" data-level="" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#data-1"><i class="fa fa-check"></i>Data</a></li>
<li class="chapter" data-level="" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#multiple-item-psychological-measures-1"><i class="fa fa-check"></i>Multiple-item Psychological Measures</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="reproducibleresearch.html"><a href="reproducibleresearch.html#final-rating"><i class="fa fa-check"></i><b>2.3</b> Final rating</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="datanalysis.html"><a href="datanalysis.html"><i class="fa fa-check"></i><b>3</b> Analyzing and visualizing data</a>
<ul>
<li class="chapter" data-level="3.1" data-path="datanalysis.html"><a href="datanalysis.html#preparing-the-data"><i class="fa fa-check"></i><b>3.1</b> Preparing the data</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="datanalysis.html"><a href="datanalysis.html#inspecting-the-data-in-r"><i class="fa fa-check"></i><b>3.1.1</b> Inspecting the data in R</a></li>
<li class="chapter" data-level="3.1.2" data-path="datanalysis.html"><a href="datanalysis.html#fixing-the-characterdouble-issue"><i class="fa fa-check"></i><b>3.1.2</b> Fixing the character/double issue</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="datanalysis.html"><a href="datanalysis.html#analyzing-and-visualizing-the-data"><i class="fa fa-check"></i><b>3.2</b> Analyzing and visualizing the data</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="datanalysis.html"><a href="datanalysis.html#plan-for-ic50ed50-visualization"><i class="fa fa-check"></i><b>3.2.1</b> Plan for IC50/ED50 visualization</a></li>
<li class="chapter" data-level="3.2.2" data-path="datanalysis.html"><a href="datanalysis.html#normalizing-the-data"><i class="fa fa-check"></i><b>3.2.2</b> Normalizing the data</a></li>
<li class="chapter" data-level="3.2.3" data-path="datanalysis.html"><a href="datanalysis.html#creating-a-legend"><i class="fa fa-check"></i><b>3.2.3</b> Creating a legend</a></li>
<li class="chapter" data-level="3.2.4" data-path="datanalysis.html"><a href="datanalysis.html#generating-the-normalized-plot"><i class="fa fa-check"></i><b>3.2.4</b> Generating the normalized plot</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="drugscreening.html"><a href="drugscreening.html"><i class="fa fa-check"></i><b>4</b> Drug Screening</a></li>
<li class="chapter" data-level="5" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html"><i class="fa fa-check"></i><b>5</b> Working with relational databases</a>
<ul>
<li class="chapter" data-level="5.1" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#preparing-the-data-1"><i class="fa fa-check"></i><b>5.1</b> Preparing the data</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#introduction"><i class="fa fa-check"></i><b>5.1.1</b> Introduction</a></li>
<li class="chapter" data-level="5.1.2" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#importing-the-data-and-making-it-tidy"><i class="fa fa-check"></i><b>5.1.2</b> Importing the data and making it tidy</a></li>
<li class="chapter" data-level="5.1.3" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#connecting-to-the-database"><i class="fa fa-check"></i><b>5.1.3</b> Connecting to the database</a></li>
<li class="chapter" data-level="5.1.4" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#exporting-the-tidy-dataframes-to-the-database"><i class="fa fa-check"></i><b>5.1.4</b> Exporting the tidy dataframes to the database</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#inspecting-the-data"><i class="fa fa-check"></i><b>5.2</b> Inspecting the data</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#inspecting-the-data-using-sql"><i class="fa fa-check"></i><b>5.2.1</b> Inspecting the data using SQL</a></li>
<li class="chapter" data-level="5.2.2" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#inspecting-the-data-using-r"><i class="fa fa-check"></i><b>5.2.2</b> Inspecting the data using R</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#combining-the-dataframes"><i class="fa fa-check"></i><b>5.3</b> Combining the dataframes</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#preparing-the-tidy-dataframes"><i class="fa fa-check"></i><b>5.3.1</b> Preparing the tidy dataframes</a></li>
<li class="chapter" data-level="5.3.2" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#joining-the-dataframes-together"><i class="fa fa-check"></i><b>5.3.2</b> Joining the dataframes together</a></li>
<li class="chapter" data-level="5.3.3" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#loading-the-combined-tables-into-r"><i class="fa fa-check"></i><b>5.3.3</b> Loading the combined tables into R</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#visualizing-the-data"><i class="fa fa-check"></i><b>5.4</b> Visualizing the data</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#normalizing-the-data-1"><i class="fa fa-check"></i><b>5.4.1</b> Normalizing the data</a></li>
<li class="chapter" data-level="5.4.2" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#calculating-descriptive-statistics"><i class="fa fa-check"></i><b>5.4.2</b> Calculating descriptive statistics</a></li>
<li class="chapter" data-level="5.4.3" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#visualizing-correlation"><i class="fa fa-check"></i><b>5.4.3</b> Visualizing correlation</a></li>
<li class="chapter" data-level="5.4.4" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#visualizing-the-dengue-data"><i class="fa fa-check"></i><b>5.4.4</b> Visualizing the dengue data</a></li>
<li class="chapter" data-level="5.4.5" data-path="working-with-relational-databases.html"><a href="working-with-relational-databases.html#visualizing-the-flu-data"><i class="fa fa-check"></i><b>5.4.5</b> Visualizing the flu data</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="parameterized.html"><a href="parameterized.html"><i class="fa fa-check"></i><b>6</b> Parameterized report on COVID-19</a>
<ul>
<li class="chapter" data-level="6.1" data-path="parameterized.html"><a href="parameterized.html#parameterized-reports"><i class="fa fa-check"></i><b>6.1</b> Parameterized reports</a></li>
<li class="chapter" data-level="6.2" data-path="parameterized.html"><a href="parameterized.html#the-parameterized-covid-19-report"><i class="fa fa-check"></i><b>6.2</b> The parameterized COVID-19 report</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="parameterized.html"><a href="parameterized.html#the-parameters"><i class="fa fa-check"></i><b>6.2.1</b> The parameters</a></li>
<li class="chapter" data-level="6.2.2" data-path="parameterized.html"><a href="parameterized.html#rendering-the-report"><i class="fa fa-check"></i><b>6.2.2</b> Rendering the report</a></li>
<li class="chapter" data-level="6.2.3" data-path="parameterized.html"><a href="parameterized.html#using-different-parameters"><i class="fa fa-check"></i><b>6.2.3</b> Using different parameters</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="vignette.html"><a href="vignette.html"><i class="fa fa-check"></i><b>7</b> R package vignette</a>
<ul>
<li class="chapter" data-level="7.1" data-path="vignette.html"><a href="vignette.html#creating-the-package"><i class="fa fa-check"></i><b>7.1</b> Creating the package</a></li>
<li class="chapter" data-level="7.2" data-path="vignette.html"><a href="vignette.html#installing-and-loading-the-package"><i class="fa fa-check"></i><b>7.2</b> Installing and loading the package</a></li>
<li class="chapter" data-level="7.3" data-path="vignette.html"><a href="vignette.html#package-contents"><i class="fa fa-check"></i><b>7.3</b> Package contents</a>
<ul>
<li class="chapter" data-level="7.3.1" data-path="vignette.html"><a href="vignette.html#dataset-measlesdata"><i class="fa fa-check"></i><b>7.3.1</b> Dataset measlesdata</a></li>
<li class="chapter" data-level="7.3.2" data-path="vignette.html"><a href="vignette.html#function-datasummarizer"><i class="fa fa-check"></i><b>7.3.2</b> Function dataSummarizer()</a></li>
<li class="chapter" data-level="7.3.3" data-path="vignette.html"><a href="vignette.html#function-illnessplot"><i class="fa fa-check"></i><b>7.3.3</b> Function illnessPlot()</a></li>
<li class="chapter" data-level="7.3.4" data-path="vignette.html"><a href="vignette.html#function-fastaimporter.r"><i class="fa fa-check"></i><b>7.3.4</b> Function fastaImporter.R</a></li>
<li class="chapter" data-level="7.3.5" data-path="vignette.html"><a href="vignette.html#function-gotermanalysis"><i class="fa fa-check"></i><b>7.3.5</b> Function gotermAnalysis()</a></li>
<li class="chapter" data-level="7.3.6" data-path="vignette.html"><a href="vignette.html#function-gotermplot"><i class="fa fa-check"></i><b>7.3.6</b> Function gotermPlot()</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="vignette.html"><a href="vignette.html#additional-files"><i class="fa fa-check"></i><b>7.4</b> Additional files</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="primates.html"><a href="primates.html"><i class="fa fa-check"></i><b>8</b> Generating a phylogenetic tree in R</a>
<ul>
<li class="chapter" data-level="8.1" data-path="primates.html"><a href="primates.html#the-plan"><i class="fa fa-check"></i><b>8.1</b> The plan</a></li>
<li class="chapter" data-level="8.2" data-path="primates.html"><a href="primates.html#preparatory-research"><i class="fa fa-check"></i><b>8.2</b> Preparatory research</a></li>
<li class="chapter" data-level="8.3" data-path="primates.html"><a href="primates.html#getting-started"><i class="fa fa-check"></i><b>8.3</b> Getting started</a></li>
<li class="chapter" data-level="8.4" data-path="primates.html"><a href="primates.html#generating-the-first-tree"><i class="fa fa-check"></i><b>8.4</b> Generating the first tree</a>
<ul>
<li class="chapter" data-level="8.4.1" data-path="primates.html"><a href="primates.html#obtaining-fasta-sequences"><i class="fa fa-check"></i><b>8.4.1</b> Obtaining FASTA sequences</a></li>
<li class="chapter" data-level="8.4.2" data-path="primates.html"><a href="primates.html#performing-multiple-sequence-alignment"><i class="fa fa-check"></i><b>8.4.2</b> Performing multiple sequence alignment</a></li>
<li class="chapter" data-level="8.4.3" data-path="primates.html"><a href="primates.html#importing-the-aligned-dna"><i class="fa fa-check"></i><b>8.4.3</b> Importing the aligned dna</a></li>
<li class="chapter" data-level="8.4.4" data-path="primates.html"><a href="primates.html#calculating-the-genetic-distances-between-the-sequences-and-drawing-a-tree"><i class="fa fa-check"></i><b>8.4.4</b> Calculating the genetic distances between the sequences and drawing a tree</a></li>
</ul></li>
<li class="chapter" data-level="8.5" data-path="primates.html"><a href="primates.html#different-types-of-phylogenetic-models"><i class="fa fa-check"></i><b>8.5</b> Different types of phylogenetic models</a>
<ul>
<li class="chapter" data-level="8.5.1" data-path="primates.html"><a href="primates.html#finding-the-best-model"><i class="fa fa-check"></i><b>8.5.1</b> Finding the best model</a></li>
<li class="chapter" data-level="8.5.2" data-path="primates.html"><a href="primates.html#generating-a-maximum-likelihood-tree"><i class="fa fa-check"></i><b>8.5.2</b> Generating a maximum likelihood tree</a></li>
<li class="chapter" data-level="8.5.3" data-path="primates.html"><a href="primates.html#visualizing-the-maximum-likelihood-cladogram"><i class="fa fa-check"></i><b>8.5.3</b> Visualizing the maximum likelihood cladogram</a></li>
<li class="chapter" data-level="8.5.4" data-path="primates.html"><a href="primates.html#rooting-the-tree"><i class="fa fa-check"></i><b>8.5.4</b> Rooting the tree</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="vignette" class="section level1" number="7">
<h1><span class="header-section-number">7</span> R package vignette</h1>
<hr />
<div id="creating-the-package" class="section level2" number="7.1">
<h2><span class="header-section-number">7.1</span> Creating the package</h2>
<p>I have created a package to help me with data analysis. My package {dsfbnine} was created with the help of the {devtools} <span class="citation">(<a href="#ref-wickhamDevtoolsToolsMake2022" role="doc-biblioref">Hadley Wickham et al. 2022</a>)</span> and {usethis} <span class="citation">(<a href="#ref-wickhamUsethisAutomatePackage2022" role="doc-biblioref">Hadley Wickham, Bryan, and Barrett 2022</a>)</span> packages. The general frame was made using function from these packages, like <code>usethis::create_package</code> and <code>devtools::document()</code>.</p>
<p>The functions included in this package are specific to my own workflow, most were written by looking at my own duplicate code. The main use of the functions are to make my code tidier and help me work faster, since I often do things like plotting illness cases for example. The rest of this chapter will be the vignette that comes with the package.</p>
<hr />
</div>
<div id="installing-and-loading-the-package" class="section level2" number="7.2">
<h2><span class="header-section-number">7.2</span> Installing and loading the package</h2>
<div class="sourceCode" id="cb77"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb77-1"><a href="vignette.html#cb77-1" aria-hidden="true" tabindex="-1"></a>devtools<span class="sc">::</span><span class="fu">install_github</span>(<span class="st">&quot;nineluijendijk/dsfbnine&quot;</span>)</span>
<span id="cb77-2"><a href="vignette.html#cb77-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(dsfbnine)</span></code></pre></div>
</div>
<div id="package-contents" class="section level2" number="7.3">
<h2><span class="header-section-number">7.3</span> Package contents</h2>
<div id="dataset-measlesdata" class="section level3" number="7.3.1">
<h3><span class="header-section-number">7.3.1</span> Dataset measlesdata</h3>
<p>The <a href="https://github.com/nineluijendijk/dsfbnine">{dsfbnine} package</a> contains some functions to help me with my data analysis and one cleaned dataset <code>measlesdata</code>. It contains data on measles cases and population numbers in EU countries from 2018 and 2019. The dataset loads as a dataframe with 744 rows and 8 variables. It is available via lazy-loading, meaning columns can be accessed as follows:</p>
<div class="sourceCode" id="cb78"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb78-1"><a href="vignette.html#cb78-1" aria-hidden="true" tabindex="-1"></a>measlesdata<span class="sc">$</span>cases <span class="sc">%&gt;%</span> <span class="fu">head</span>(<span class="at">n =</span> <span class="dv">20</span>)</span></code></pre></div>
<pre><code>##  [1]  7  5 15 17 12  6  1  3  6  0  4  1  1  6  9 11 24 22  3 11</code></pre>
</div>
<div id="function-datasummarizer" class="section level3" number="7.3.2">
<h3><span class="header-section-number">7.3.2</span> Function dataSummarizer()</h3>
<p>The function <code>dataSummarizer()</code> calculates the sum, mean, median and standard deviation of data, grouped by a given argument:</p>
<div class="sourceCode" id="cb80"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb80-1"><a href="vignette.html#cb80-1" aria-hidden="true" tabindex="-1"></a><span class="fu">dataSummarizer</span>(measlesdata, <span class="at">groupBy =</span> <span class="fu">c</span>(<span class="st">&quot;CountryName&quot;</span>, <span class="st">&quot;year&quot;</span>)) <span class="sc">%&gt;%</span> <span class="fu">head</span>(<span class="at">n =</span> <span class="dv">10</span>)</span></code></pre></div>
<pre><code>## # A tibble: 10 × 6
## # Groups:   CountryName [7]
##    CountryName year  total  mean median standarddeviation
##    &lt;chr&gt;       &lt;chr&gt; &lt;dbl&gt; &lt;dbl&gt;  &lt;dbl&gt;             &lt;dbl&gt;
##  1 EU/EEA      2019  13207 1101. 1012.              783. 
##  2 EU/EEA      2018  12352 1029.  959               687. 
##  3 France      2018   2913  243.  136               252. 
##  4 France      2019   2636  220.  166.              176. 
##  5 Italy       2018   2517  210.  170               155. 
##  6 Greece      2018   2293  191.   96.5             211. 
##  7 Romania     2019   1706  142.  111                72.8
##  8 Italy       2019   1626  136.  162               106. 
##  9 Poland      2019   1423  119.   82.5             121. 
## 10 Bulgaria    2019   1235  103.   46.5             110.</code></pre>
</div>
<div id="function-illnessplot" class="section level3" number="7.3.3">
<h3><span class="header-section-number">7.3.3</span> Function illnessPlot()</h3>
<p>The function <code>illnessPlot()</code> generates a graph showing illness numbers, specifically COVID-19 numbers from an <a href="https://www.ecdc.europa.eu/en/covid-19/data">ECDC</a> dataset, used in <a href="https://nineluijendijk.github.io/parameterized.html">Chapter 6</a> of my portfolio. The same code was used to generate 4 different plots, to fix this issue <code>illnessPlot()</code> was created.</p>
<div class="sourceCode" id="cb82"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb82-1"><a href="vignette.html#cb82-1" aria-hidden="true" tabindex="-1"></a>coviddata <span class="ot">&lt;-</span> <span class="fu">read.csv</span>(here<span class="sc">::</span><span class="fu">here</span>(<span class="st">&quot;data/datacovid.csv&quot;</span>)) <span class="co">#load the data</span></span>
<span id="cb82-2"><a href="vignette.html#cb82-2" aria-hidden="true" tabindex="-1"></a><span class="fu">illnessPlot</span>(coviddata, <span class="at">countries =</span> <span class="fu">c</span>(<span class="st">&quot;Netherlands&quot;</span>, <span class="st">&quot;Belgium&quot;</span>, <span class="st">&quot;France&quot;</span>), <span class="at">years =</span> <span class="dv">2020</span><span class="sc">:</span><span class="dv">2022</span>, <span class="at">parameter =</span> cases)</span></code></pre></div>
<div class="figure"><span style="display:block;" id="fig:illnessPlotexample"></span>
<img src="portfolio_book_files/figure-html/illnessPlotexample-1.png" alt="Example illness plot." width="691.2" />
<p class="caption">
Figure 7.1: Example illness plot.
</p>
</div>
</div>
<div id="function-fastaimporter.r" class="section level3" number="7.3.4">
<h3><span class="header-section-number">7.3.4</span> Function fastaImporter.R</h3>
<p>The function <code>fastaImporter()</code> can be used to import FASTA sequences by entering their GenBank Identifiers. The imported sequences will be stored in an object of class “DNAbin”.</p>
<div class="sourceCode" id="cb83"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb83-1"><a href="vignette.html#cb83-1" aria-hidden="true" tabindex="-1"></a>dna <span class="ot">&lt;-</span> <span class="fu">fastaImporter</span>(here<span class="sc">::</span><span class="fu">here</span>(<span class="st">&quot;data/species.txt&quot;</span>))</span>
<span id="cb83-2"><a href="vignette.html#cb83-2" aria-hidden="true" tabindex="-1"></a>dna</span></code></pre></div>
<pre><code>## 16 DNA sequences in binary format stored in a list.
## 
## Mean sequence length: 16523.81 
##    Shortest sequence: 16264 
##     Longest sequence: 17062 
## 
## Labels:
## KF914213.1 Gorilla beringei graueri mitochondrion, complete ...
## NC_011120.1 Gorilla gorilla gorilla mitochondrion, complete ...
## GU170821.1 Homo sapiens isolate NS07 mitochondrion, complete...
## HM015213.1 Pan paniscus isolate PP25 mitochondrion, complete...
## NC_001643.1 Pan troglodytes mitochondrion, complete genome
## NC_002083.1 Pongo abelii mitochondrion, complete genome
## ...
## 
## Base composition:
##     a     c     g     t 
## 0.314 0.299 0.131 0.256 
## (Total: 264.38 kb)</code></pre>
</div>
<div id="function-gotermanalysis" class="section level3" number="7.3.5">
<h3><span class="header-section-number">7.3.5</span> Function gotermAnalysis()</h3>
<p>The function <code>gotermAnalysis()</code> was written specifically to help with my analysis of RNA-sequencing data, which is why there are many different parameters to be set: changing just 1 can completely change the results (seen in the code chunk below). The function performs a GO-term enrichment analysis. The output of this function can be used in the next function <code>gotermPlot()</code> as well.</p>
<div class="sourceCode" id="cb85"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb85-1"><a href="vignette.html#cb85-1" aria-hidden="true" tabindex="-1"></a>DESeqresults <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(here<span class="sc">::</span><span class="fu">here</span>(<span class="st">&quot;data/dge_results&quot;</span>)) <span class="co">#load the data object of class &quot;DESeqResults&quot;</span></span>
<span id="cb85-2"><a href="vignette.html#cb85-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb85-3"><a href="vignette.html#cb85-3" aria-hidden="true" tabindex="-1"></a>gotermAnalysis_results <span class="ot">&lt;-</span> <span class="fu">gotermAnalysis</span>(DESeqresults)</span></code></pre></div>
</div>
<div id="function-gotermplot" class="section level3" number="7.3.6">
<h3><span class="header-section-number">7.3.6</span> Function gotermPlot()</h3>
<p>The function <code>gotermPlot()</code> is an extension to the previous <code>gotermAnalysis()</code> function, generating a figure containing the most abundant GO-terms. To save computing time, the <code>gotermAnalysis()</code> function will not be ran. This means its output has to be imported for the <code>gotermPlot()</code> function:</p>
<div class="sourceCode" id="cb86"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb86-1"><a href="vignette.html#cb86-1" aria-hidden="true" tabindex="-1"></a>gotermAnalysis_results <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(here<span class="sc">::</span><span class="fu">here</span>(<span class="st">&quot;data/gotermAnalysis_results&quot;</span>))</span>
<span id="cb86-2"><a href="vignette.html#cb86-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb86-3"><a href="vignette.html#cb86-3" aria-hidden="true" tabindex="-1"></a><span class="fu">gotermPlot</span>(gotermAnalysis_results, <span class="at">topamount =</span> <span class="dv">10</span>, <span class="at">plot_title =</span> <span class="st">&quot;Top 10 upregulated GO-terms&quot;</span>)</span></code></pre></div>
<div class="figure"><span style="display:block;" id="fig:gotermPlotexample"></span>
<img src="portfolio_book_files/figure-html/gotermPlotexample-1.png" alt="Example GO-term plot." width="691.2" />
<p class="caption">
Figure 7.2: Example GO-term plot.
</p>
</div>
</div>
</div>
<div id="additional-files" class="section level2" number="7.4">
<h2><span class="header-section-number">7.4</span> Additional files</h2>
<p>Additional files can be found in the <code>/inst/extdata/</code> directory. It consist of files used to build the vignette, (raw) data for analysis/plotting, example files and an R script that shows how the dataset <code>measlesdata</code> was cleaned.</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-wickhamUsethisAutomatePackage2022" class="csl-entry">
Wickham, Hadley, Jennifer Bryan, and Malcom Barrett. 2022. <span>“Usethis: Automate Package and Project Setup.”</span> <a href="https://usethis.r-lib.org, https://github.com/r-lib/usethis">https://usethis.r-lib.org, https://github.com/r-lib/usethis</a>.
</div>
<div id="ref-wickhamDevtoolsToolsMake2022" class="csl-entry">
Wickham, Hadley, Jim Hester, Winston Chang, and Jennifer Bryan. 2022. <span>“Devtools: Tools to Make Developing R Packages Easier.”</span> <a href="https://devtools.r-lib.org/, https://github.com/r-lib/devtools">https://devtools.r-lib.org/, https://github.com/r-lib/devtools</a>.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="parameterized.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="primates.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
}
});
});
</script>

</body>

</html>
